2020/02/27 10:35:40 [error] 6#6: *2 open() "/usr/share/nginx/html/favicon.ico" failed (2: No such file or directory), client: 192.168.224.1, server: localhost, request: "GET /favicon.ico HTTP/1.0", host: "mills.prod"
2020/02/27 10:37:10 [error] 6#6: *9 open() "/usr/share/nginx/html/favicon.ico" failed (2: No such file or directory), client: 192.168.224.1, server: localhost, request: "GET /favicon.ico HTTP/1.0", host: "mills.prod"
2020/02/27 10:50:13 [error] 6#6: *3 open() "/usr/share/nginx/html/favicon.ico" failed (2: No such file or directory), client: 192.168.240.1, server: localhost, request: "GET /favicon.ico HTTP/1.0", host: "mills.prod"
2020/02/27 10:58:38 [error] 6#6: *339 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.240.1, server: , request: "GET / HTTP/1.0", upstream: "http://192.168.240.3:5601/", host: "mills.prod"
2020/02/27 10:58:39 [error] 6#6: *341 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.240.1, server: , request: "GET / HTTP/1.0", upstream: "http://192.168.240.3:5601/", host: "mills.prod"
2020/02/27 10:58:44 [error] 6#6: *345 recv() failed (104: Connection reset by peer) while reading response header from upstream, client: 192.168.240.1, server: , request: "GET /app/kibana HTTP/1.0", upstream: "http://192.168.240.3:5601/app/kibana", host: "mills.prod"
2020/02/27 10:58:44 [error] 6#6: *347 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.240.1, server: , request: "GET /app/kibana HTTP/1.0", upstream: "http://192.168.240.3:5601/app/kibana", host: "mills.prod"
2020/02/27 11:00:59 [error] 6#6: *1097 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.240.1, server: , request: "GET /app/kibana HTTP/1.0", upstream: "http://192.168.240.3:5601/app/kibana", host: "mills.prod"
2020/02/27 11:00:59 [error] 6#6: *1099 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.240.1, server: , request: "GET /app/kibana HTTP/1.0", upstream: "http://192.168.240.3:5601/app/kibana", host: "mills.prod"
2020/02/27 11:01:00 [error] 6#6: *1101 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.240.1, server: , request: "GET /app/kibana HTTP/1.0", upstream: "http://192.168.240.3:5601/app/kibana", host: "mills.prod"
2020/02/27 11:01:01 [error] 6#6: *1103 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.240.1, server: , request: "GET /app/kibana HTTP/1.0", upstream: "http://192.168.240.3:5601/app/kibana", host: "mills.prod"
2020/02/27 11:01:04 [error] 6#6: *1105 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.240.1, server: , request: "GET / HTTP/1.0", upstream: "http://192.168.240.3:5601/", host: "mills.prod"
2020/02/27 11:01:23 [error] 6#6: *1107 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.240.1, server: , request: "GET / HTTP/1.0", upstream: "http://192.168.240.3:5601/", host: "mills.prod"
2020/02/27 11:01:24 [error] 6#6: *1109 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.240.1, server: , request: "GET / HTTP/1.0", upstream: "http://192.168.240.3:5601/", host: "mills.prod"
2020/02/27 11:01:24 [error] 6#6: *1111 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.240.1, server: , request: "GET / HTTP/1.0", upstream: "http://192.168.240.3:5601/", host: "mills.prod"
2020/02/27 11:05:21 [error] 6#6: *1 connect() failed (113: No route to host) while connecting to upstream, client: 172.18.0.1, server: , request: "GET / HTTP/1.0", upstream: "http://172.18.0.4:5601/", host: "mills.prod"
2020/02/27 11:05:24 [error] 6#6: *3 connect() failed (113: No route to host) while connecting to upstream, client: 172.18.0.1, server: , request: "GET / HTTP/1.0", upstream: "http://172.18.0.4:5601/", host: "mills.prod"
2020/02/27 11:15:43 [error] 6#6: *5 connect() failed (111: Connection refused) while connecting to upstream, client: 172.18.0.1, server: , request: "GET / HTTP/1.0", upstream: "http://172.18.0.4:5601/", host: "mills.prod"
2020/02/27 11:15:46 [error] 6#6: *7 connect() failed (111: Connection refused) while connecting to upstream, client: 172.18.0.1, server: , request: "GET / HTTP/1.0", upstream: "http://172.18.0.4:5601/", host: "mills.prod"
2020/02/27 13:24:48 [error] 7#7: *1 connect() failed (113: No route to host) while connecting to upstream, client: 172.19.0.1, server: , request: "GET /app/kibana HTTP/1.0", upstream: "http://172.19.0.5:5601/app/kibana", host: "mills.prod"
2020/02/27 13:24:55 [error] 7#7: *4 connect() failed (113: No route to host) while connecting to upstream, client: 172.19.0.1, server: , request: "GET / HTTP/1.0", upstream: "http://172.19.0.5:5601/", host: "mills.prod"
2020/02/27 13:25:05 [error] 7#7: *6 connect() failed (113: No route to host) while connecting to upstream, client: 172.19.0.1, server: , request: "GET /app/kibana HTTP/1.0", upstream: "http://172.19.0.5:5601/app/kibana", host: "mills.prod"
2020/02/27 13:25:43 [error] 7#7: *8 connect() failed (113: No route to host) while connecting to upstream, client: 172.19.0.1, server: , request: "GET / HTTP/1.0", upstream: "http://172.19.0.5:5601/", host: "mills.prod"
2020/02/27 13:27:05 [error] 7#7: *12 connect() failed (113: No route to host) while connecting to upstream, client: 172.19.0.1, server: , request: "GET / HTTP/1.0", upstream: "http://172.19.0.5:5601/", host: "mills.prod"
2020/02/27 13:27:11 [error] 7#7: *15 connect() failed (113: No route to host) while connecting to upstream, client: 172.19.0.1, server: , request: "GET / HTTP/1.0", upstream: "http://172.19.0.5:5601/", host: "mills.prod"
2020/02/27 13:27:47 [error] 7#7: *17 connect() failed (113: No route to host) while connecting to upstream, client: 172.19.0.1, server: , request: "GET / HTTP/1.0", upstream: "http://172.19.0.5:5601/", host: "mills.prod"
2020/02/27 13:28:56 [error] 7#7: *19 connect() failed (113: No route to host) while connecting to upstream, client: 172.19.0.1, server: , request: "GET / HTTP/1.0", upstream: "http://172.19.0.5:5601/", host: "mills.prod"
2020/03/02 14:05:51 [error] 6#6: *5 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.200.55, server: , request: "GET / HTTP/1.1", upstream: "http://172.22.0.6:5601/", host: "lamas.int"
2020/03/02 15:27:50 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.200.68, server: , request: "GET /app/kibana HTTP/1.1", upstream: "http://172.25.0.2:5601/app/kibana", host: "lamas.int"
2020/03/02 15:28:06 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.200.68, server: , request: "GET / HTTP/1.1", upstream: "http://172.25.0.2:5601/", host: "lamas.int"
2020/03/02 15:28:08 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.200.68, server: , request: "GET / HTTP/1.1", upstream: "http://172.25.0.2:5601/", host: "lamas.int"
2020/03/02 15:28:08 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.200.68, server: , request: "GET / HTTP/1.1", upstream: "http://172.25.0.2:5601/", host: "lamas.int"
2020/03/02 15:28:08 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.200.68, server: , request: "GET / HTTP/1.1", upstream: "http://172.25.0.2:5601/", host: "lamas.int"
2020/03/02 15:28:09 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.200.68, server: , request: "GET / HTTP/1.1", upstream: "http://172.25.0.2:5601/", host: "lamas.int"
2020/03/02 15:30:36 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.200.68, server: , request: "GET / HTTP/1.1", upstream: "http://172.26.0.6:5601/", host: "lamas.int"
2020/03/02 15:30:40 [error] 6#6: *3 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.200.55, server: , request: "GET / HTTP/1.1", upstream: "http://172.26.0.6:5601/", host: "lamas.int"
2020/03/02 15:30:42 [error] 6#6: *3 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.200.55, server: , request: "GET / HTTP/1.1", upstream: "http://172.26.0.6:5601/", host: "lamas.int"
2020/03/02 15:30:43 [error] 6#6: *3 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.200.55, server: , request: "GET / HTTP/1.1", upstream: "http://172.26.0.6:5601/", host: "lamas.int"
2020/03/02 15:30:43 [error] 6#6: *3 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.200.55, server: , request: "GET / HTTP/1.1", upstream: "http://172.26.0.6:5601/", host: "lamas.int"
2020/03/02 15:30:43 [error] 6#6: *3 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.200.55, server: , request: "GET / HTTP/1.1", upstream: "http://172.26.0.6:5601/", host: "lamas.int"
2020/03/02 15:30:46 [error] 6#6: *10 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.200.55, server: , request: "GET /app/kibana HTTP/1.1", upstream: "http://172.26.0.6:5601/app/kibana", host: "lamas.int"
2020/03/02 15:30:49 [error] 6#6: *10 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.200.55, server: , request: "GET / HTTP/1.1", upstream: "http://172.26.0.6:5601/", host: "lamas.int"
2020/03/02 15:30:50 [error] 6#6: *10 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.200.55, server: , request: "GET / HTTP/1.1", upstream: "http://172.26.0.6:5601/", host: "lamas.int"
2020/03/02 15:31:17 [error] 6#6: *14 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.200.55, server: , request: "GET / HTTP/1.1", upstream: "http://172.26.0.6:5601/", host: "lamas.int"
2020/03/02 15:31:18 [error] 6#6: *14 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.200.55, server: , request: "GET / HTTP/1.1", upstream: "http://172.26.0.6:5601/", host: "lamas.int"
2020/03/02 15:31:19 [error] 6#6: *14 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.200.55, server: , request: "GET / HTTP/1.1", upstream: "http://172.26.0.6:5601/", host: "lamas.int"
2020/03/02 15:31:19 [error] 6#6: *14 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.200.55, server: , request: "GET / HTTP/1.1", upstream: "http://172.26.0.6:5601/", host: "lamas.int"
2020/03/02 15:32:23 [error] 6#6: *19 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.200.68, server: , request: "GET / HTTP/1.1", upstream: "http://172.26.0.6:5601/", host: "lamas.int"
2020/03/02 15:32:28 [error] 6#6: *21 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.200.55, server: , request: "GET / HTTP/1.1", upstream: "http://172.26.0.6:5601/", host: "lamas.int"
2020/03/02 15:32:31 [error] 6#6: *21 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.200.55, server: , request: "GET / HTTP/1.1", upstream: "http://172.26.0.6:5601/", host: "lamas.int"
2020/03/02 15:32:32 [error] 6#6: *21 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.200.55, server: , request: "GET / HTTP/1.1", upstream: "http://172.26.0.6:5601/", host: "lamas.int"
2020/03/02 15:32:33 [error] 6#6: *21 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.200.55, server: , request: "GET / HTTP/1.1", upstream: "http://172.26.0.6:5601/", host: "lamas.int"
2020/03/02 15:32:40 [error] 6#6: *21 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.200.55, server: , request: "GET / HTTP/1.1", upstream: "http://172.26.0.6:5601/", host: "lamas.int"
2020/03/02 15:32:43 [error] 6#6: *21 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.200.55, server: , request: "GET / HTTP/1.1", upstream: "http://172.26.0.6:5601/", host: "lamas.int"
2020/03/02 15:33:06 [error] 6#6: *21 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.200.55, server: , request: "GET / HTTP/1.1", upstream: "http://172.26.0.6:5601/", host: "lamas.int"
2020/03/02 15:33:06 [error] 6#6: *21 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.200.55, server: , request: "GET / HTTP/1.1", upstream: "http://172.26.0.6:5601/", host: "lamas.int"
2020/03/02 15:33:06 [error] 6#6: *21 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.200.55, server: , request: "GET / HTTP/1.1", upstream: "http://172.26.0.6:5601/", host: "lamas.int"
2020/03/02 15:33:07 [error] 6#6: *21 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.200.55, server: , request: "GET / HTTP/1.1", upstream: "http://172.26.0.6:5601/", host: "lamas.int"
2020/03/02 15:33:07 [error] 6#6: *21 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.200.55, server: , request: "GET / HTTP/1.1", upstream: "http://172.26.0.6:5601/", host: "lamas.int"
2020/03/02 15:33:07 [error] 6#6: *21 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.200.55, server: , request: "GET / HTTP/1.1", upstream: "http://172.26.0.6:5601/", host: "lamas.int"
2020/03/02 15:33:07 [error] 6#6: *21 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.200.55, server: , request: "GET / HTTP/1.1", upstream: "http://172.26.0.6:5601/", host: "lamas.int"
2020/03/02 15:33:08 [error] 6#6: *21 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.200.55, server: , request: "GET / HTTP/1.1", upstream: "http://172.26.0.6:5601/", host: "lamas.int"
2020/03/02 15:58:53 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.200.68, server: , request: "POST /api/ui_metric/report HTTP/1.1", upstream: "http://172.27.0.5:5601/api/ui_metric/report", host: "lamas.int", referrer: "http://lamas.int/app/kibana"
2020/03/02 15:58:53 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.200.68, server: , request: "GET /app/kibana HTTP/1.1", upstream: "http://172.27.0.5:5601/app/kibana", host: "lamas.int"
2020/03/02 15:58:55 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.200.68, server: , request: "GET /app/kibana HTTP/1.1", upstream: "http://172.27.0.5:5601/app/kibana", host: "lamas.int"
2020/03/02 15:58:55 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.200.68, server: , request: "GET /app/kibana HTTP/1.1", upstream: "http://172.27.0.5:5601/app/kibana", host: "lamas.int"
2020/03/02 15:58:55 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.200.68, server: , request: "GET /app/kibana HTTP/1.1", upstream: "http://172.27.0.5:5601/app/kibana", host: "lamas.int"
2020/03/02 15:58:55 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.200.68, server: , request: "GET /app/kibana HTTP/1.1", upstream: "http://172.27.0.5:5601/app/kibana", host: "lamas.int"
2020/03/02 15:58:55 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.200.68, server: , request: "GET /app/kibana HTTP/1.1", upstream: "http://172.27.0.5:5601/app/kibana", host: "lamas.int"
2020/03/02 15:58:56 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.200.68, server: , request: "GET /app/kibana HTTP/1.1", upstream: "http://172.27.0.5:5601/app/kibana", host: "lamas.int"
2020/03/02 15:58:57 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.200.68, server: , request: "GET /app/kibana HTTP/1.1", upstream: "http://172.27.0.5:5601/app/kibana", host: "lamas.int"
2020/03/02 15:58:58 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.200.68, server: , request: "GET /app/kibana HTTP/1.1", upstream: "http://172.27.0.5:5601/app/kibana", host: "lamas.int"
2020/03/02 15:58:59 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.200.68, server: , request: "GET /app/kibana HTTP/1.1", upstream: "http://172.27.0.5:5601/app/kibana", host: "lamas.int"
2020/03/02 15:59:00 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.200.68, server: , request: "GET /app/kibana HTTP/1.1", upstream: "http://172.27.0.5:5601/app/kibana", host: "lamas.int"
2020/03/02 15:59:01 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.200.68, server: , request: "GET /app/kibana HTTP/1.1", upstream: "http://172.27.0.5:5601/app/kibana", host: "lamas.int"
2020/03/02 15:59:02 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.200.68, server: , request: "GET /app/kibana HTTP/1.1", upstream: "http://172.27.0.5:5601/app/kibana", host: "lamas.int"
2020/03/02 15:59:03 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.200.68, server: , request: "GET /app/kibana HTTP/1.1", upstream: "http://172.27.0.5:5601/app/kibana", host: "lamas.int"
2020/03/02 15:59:04 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.200.68, server: , request: "GET /app/kibana HTTP/1.1", upstream: "http://172.27.0.5:5601/app/kibana", host: "lamas.int"
2020/03/02 15:59:07 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.200.68, server: , request: "GET / HTTP/1.1", upstream: "http://172.27.0.5:5601/", host: "lamas.int"
2020/03/02 15:59:22 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.200.68, server: , request: "GET / HTTP/1.1", upstream: "http://172.27.0.5:5601/", host: "lamas.int"
2020/03/02 15:59:24 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.200.68, server: , request: "GET / HTTP/1.1", upstream: "http://172.27.0.5:5601/", host: "lamas.int"
2020/03/02 15:59:45 [error] 6#6: *27 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.200.68, server: , request: "GET / HTTP/1.1", upstream: "http://172.27.0.5:5601/", host: "lamas.int"
2020/03/02 15:59:49 [error] 6#6: *27 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.200.68, server: , request: "GET / HTTP/1.1", upstream: "http://172.27.0.5:5601/", host: "lamas.int"
2020/03/02 15:59:50 [error] 6#6: *27 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.200.68, server: , request: "GET / HTTP/1.1", upstream: "http://172.27.0.5:5601/", host: "lamas.int"
2020/03/02 15:59:50 [error] 6#6: *27 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.200.68, server: , request: "GET / HTTP/1.1", upstream: "http://172.27.0.5:5601/", host: "lamas.int"
2020/03/02 15:59:50 [error] 6#6: *27 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.200.68, server: , request: "GET / HTTP/1.1", upstream: "http://172.27.0.5:5601/", host: "lamas.int"
2020/03/02 16:00:58 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.200.68, server: , request: "GET / HTTP/1.1", upstream: "http://172.28.0.7:5601/", host: "lamas.int"
2020/03/02 16:00:58 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.200.68, server: , request: "GET / HTTP/1.1", upstream: "http://172.28.0.7:5601/", host: "lamas.int"
2020/03/02 16:00:59 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.200.68, server: , request: "GET / HTTP/1.1", upstream: "http://172.28.0.7:5601/", host: "lamas.int"
2020/03/02 16:00:59 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.200.68, server: , request: "GET / HTTP/1.1", upstream: "http://172.28.0.7:5601/", host: "lamas.int"
2020/03/02 16:00:59 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.200.68, server: , request: "GET / HTTP/1.1", upstream: "http://172.28.0.7:5601/", host: "lamas.int"
2020/03/02 16:00:59 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.200.68, server: , request: "GET / HTTP/1.1", upstream: "http://172.28.0.7:5601/", host: "lamas.int"
2020/03/02 16:00:59 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.200.68, server: , request: "GET / HTTP/1.1", upstream: "http://172.28.0.7:5601/", host: "lamas.int"
2020/03/02 16:00:59 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.200.68, server: , request: "GET / HTTP/1.1", upstream: "http://172.28.0.7:5601/", host: "lamas.int"
2020/03/02 16:01:00 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.200.68, server: , request: "GET / HTTP/1.1", upstream: "http://172.28.0.7:5601/", host: "lamas.int"
2020/03/02 16:01:00 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.200.68, server: , request: "GET / HTTP/1.1", upstream: "http://172.28.0.7:5601/", host: "lamas.int"
2020/03/02 16:01:00 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.200.68, server: , request: "GET / HTTP/1.1", upstream: "http://172.28.0.7:5601/", host: "lamas.int"
2020/03/02 16:01:01 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.200.68, server: , request: "GET / HTTP/1.1", upstream: "http://172.28.0.7:5601/", host: "lamas.int"
2020/03/02 16:01:01 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.200.68, server: , request: "GET / HTTP/1.1", upstream: "http://172.28.0.7:5601/", host: "lamas.int"
2020/03/02 16:01:01 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.200.68, server: , request: "GET / HTTP/1.1", upstream: "http://172.28.0.7:5601/", host: "lamas.int"
2020/03/02 16:01:01 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.200.68, server: , request: "GET / HTTP/1.1", upstream: "http://172.28.0.7:5601/", host: "lamas.int"
2020/03/02 16:01:01 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.200.68, server: , request: "GET / HTTP/1.1", upstream: "http://172.28.0.7:5601/", host: "lamas.int"
2020/03/02 16:01:02 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.200.68, server: , request: "GET / HTTP/1.1", upstream: "http://172.28.0.7:5601/", host: "lamas.int"
2020/03/02 16:01:02 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.200.68, server: , request: "GET / HTTP/1.1", upstream: "http://172.28.0.7:5601/", host: "lamas.int"
2020/03/02 16:01:02 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.200.68, server: , request: "GET / HTTP/1.1", upstream: "http://172.28.0.7:5601/", host: "lamas.int"
2020/03/02 16:01:02 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.200.68, server: , request: "GET / HTTP/1.1", upstream: "http://172.28.0.7:5601/", host: "lamas.int"
2020/03/02 16:01:02 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.200.68, server: , request: "GET / HTTP/1.1", upstream: "http://172.28.0.7:5601/", host: "lamas.int"
2020/03/02 16:01:22 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.200.68, server: , request: "GET / HTTP/1.1", upstream: "http://172.28.0.7:5601/", host: "lamas.int"
2020/03/02 16:01:23 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.200.68, server: , request: "GET / HTTP/1.1", upstream: "http://172.28.0.7:5601/", host: "lamas.int"
2020/03/02 16:01:24 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.200.68, server: , request: "GET / HTTP/1.1", upstream: "http://172.28.0.7:5601/", host: "lamas.int"
2020/03/02 16:01:24 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.200.68, server: , request: "GET / HTTP/1.1", upstream: "http://172.28.0.7:5601/", host: "lamas.int"
2020/03/02 16:01:25 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.200.68, server: , request: "GET / HTTP/1.1", upstream: "http://172.28.0.7:5601/", host: "lamas.int"
2020/03/02 16:01:39 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.200.68, server: , request: "GET / HTTP/1.1", upstream: "http://172.28.0.7:5601/", host: "lamas.int"
2020/03/02 16:23:29 [error] 6#6: *340 upstream prematurely closed connection while reading response header from upstream, client: 10.20.200.68, server: , request: "POST /api/ui_metric/report HTTP/1.1", upstream: "http://172.28.0.7:5601/api/ui_metric/report", host: "lamas.int", referrer: "http://lamas.int/app/kibana"
2020/03/02 16:23:40 [error] 6#6: *342 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.200.55, server: , request: "GET / HTTP/1.1", upstream: "http://172.28.0.7:5601/", host: "lamas.int"
2020/03/02 16:23:56 [error] 6#6: *342 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.200.55, server: , request: "GET / HTTP/1.1", upstream: "http://172.28.0.7:5601/", host: "lamas.int"
2020/03/02 16:23:57 [error] 6#6: *342 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.200.55, server: , request: "GET / HTTP/1.1", upstream: "http://172.28.0.7:5601/", host: "lamas.int"
2020/03/02 16:23:57 [error] 6#6: *342 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.200.55, server: , request: "GET / HTTP/1.1", upstream: "http://172.28.0.7:5601/", host: "lamas.int"
2020/03/02 16:23:57 [error] 6#6: *342 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.200.55, server: , request: "GET / HTTP/1.1", upstream: "http://172.28.0.7:5601/", host: "lamas.int"
2020/03/02 16:23:57 [error] 6#6: *342 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.200.55, server: , request: "GET / HTTP/1.1", upstream: "http://172.28.0.7:5601/", host: "lamas.int"
2020/03/02 16:23:57 [error] 6#6: *342 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.200.55, server: , request: "GET / HTTP/1.1", upstream: "http://172.28.0.7:5601/", host: "lamas.int"
2020/03/02 16:23:57 [error] 6#6: *342 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.200.55, server: , request: "GET / HTTP/1.1", upstream: "http://172.28.0.7:5601/", host: "lamas.int"
2020/03/02 16:24:24 [error] 6#6: *342 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.200.55, server: , request: "GET / HTTP/1.1", upstream: "http://172.28.0.7:5601/", host: "lamas.int"
2020/03/02 16:24:32 [error] 6#6: *342 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.200.55, server: , request: "GET / HTTP/1.1", upstream: "http://172.28.0.7:5601/", host: "lamas.int"
2020/03/02 16:24:33 [error] 6#6: *342 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.200.55, server: , request: "GET / HTTP/1.1", upstream: "http://172.28.0.7:5601/", host: "lamas.int"
2020/03/02 16:24:33 [error] 6#6: *342 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.200.55, server: , request: "GET / HTTP/1.1", upstream: "http://172.28.0.7:5601/", host: "lamas.int"
2020/03/02 16:24:33 [error] 6#6: *342 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.200.55, server: , request: "GET / HTTP/1.1", upstream: "http://172.28.0.7:5601/", host: "lamas.int"
2020/03/02 16:24:33 [error] 6#6: *342 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.200.55, server: , request: "GET / HTTP/1.1", upstream: "http://172.28.0.7:5601/", host: "lamas.int"
2020/03/02 16:24:34 [error] 6#6: *342 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.200.55, server: , request: "GET / HTTP/1.1", upstream: "http://172.28.0.7:5601/", host: "lamas.int"
2020/03/02 16:24:34 [error] 6#6: *342 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.200.55, server: , request: "GET / HTTP/1.1", upstream: "http://172.28.0.7:5601/", host: "lamas.int"
2020/03/02 16:24:40 [error] 6#6: *359 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.200.55, server: , request: "GET / HTTP/1.1", upstream: "http://172.28.0.7:5601/", host: "lamas.int"
2020/03/02 16:24:59 [error] 6#6: *362 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.200.68, server: , request: "POST /api/ui_metric/report HTTP/1.1", upstream: "http://172.28.0.7:5601/api/ui_metric/report", host: "lamas.int", referrer: "http://lamas.int/app/kibana"
2020/03/02 16:25:34 [error] 6#6: *364 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.200.55, server: , request: "GET / HTTP/1.1", upstream: "http://172.28.0.7:5601/", host: "lamas.int"
2020/03/02 16:25:38 [error] 6#6: *364 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.200.55, server: , request: "GET / HTTP/1.1", upstream: "http://172.28.0.7:5601/", host: "lamas.int"
2020/03/02 16:25:41 [error] 6#6: *364 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.200.55, server: , request: "GET / HTTP/1.1", upstream: "http://172.28.0.7:5601/", host: "lamas.int"
2020/03/02 16:25:42 [error] 6#6: *364 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.200.55, server: , request: "GET / HTTP/1.1", upstream: "http://172.28.0.7:5601/", host: "lamas.int"
2020/03/02 16:25:42 [error] 6#6: *364 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.200.55, server: , request: "GET / HTTP/1.1", upstream: "http://172.28.0.7:5601/", host: "lamas.int"
2020/03/02 16:25:42 [error] 6#6: *364 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.200.55, server: , request: "GET / HTTP/1.1", upstream: "http://172.28.0.7:5601/", host: "lamas.int"
2020/03/02 16:25:42 [error] 6#6: *371 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.200.55, server: , request: "GET / HTTP/1.1", upstream: "http://172.28.0.7:5601/", host: "lamas.int"
2020/03/02 16:25:42 [error] 6#6: *371 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.200.55, server: , request: "GET / HTTP/1.1", upstream: "http://172.28.0.7:5601/", host: "lamas.int"
2020/03/02 16:26:04 [error] 6#6: *374 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.200.55, server: , request: "GET / HTTP/1.1", upstream: "http://172.28.0.7:5601/", host: "lamas.int"
2020/03/02 16:26:07 [error] 6#6: *374 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.200.55, server: , request: "GET / HTTP/1.1", upstream: "http://172.28.0.7:5601/", host: "lamas.int"
2020/03/02 16:26:07 [error] 6#6: *374 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.200.55, server: , request: "GET / HTTP/1.1", upstream: "http://172.28.0.7:5601/", host: "lamas.int"
2020/03/03 13:01:06 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.197.35, server: , request: "GET / HTTP/1.1", upstream: "http://172.29.0.7:5601/", host: "10.20.203.100"
2020/03/03 13:01:07 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.197.35, server: , request: "GET / HTTP/1.1", upstream: "http://172.29.0.7:5601/", host: "10.20.203.100"
2020/03/03 13:01:20 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.197.35, server: , request: "GET / HTTP/1.1", upstream: "http://172.29.0.7:5601/", host: "10.20.203.100"
2020/03/03 13:01:21 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.197.35, server: , request: "GET / HTTP/1.1", upstream: "http://172.29.0.7:5601/", host: "10.20.203.100"
2020/03/03 13:01:22 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.197.35, server: , request: "GET / HTTP/1.1", upstream: "http://172.29.0.7:5601/", host: "10.20.203.100"
2020/03/03 13:01:22 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.197.35, server: , request: "GET / HTTP/1.1", upstream: "http://172.29.0.7:5601/", host: "10.20.203.100"
2020/03/03 13:01:23 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.197.35, server: , request: "GET / HTTP/1.1", upstream: "http://172.29.0.7:5601/", host: "10.20.203.100"
2020/03/03 13:01:23 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.197.35, server: , request: "GET / HTTP/1.1", upstream: "http://172.29.0.7:5601/", host: "10.20.203.100"
2020/03/03 13:01:23 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.197.35, server: , request: "GET / HTTP/1.1", upstream: "http://172.29.0.7:5601/", host: "10.20.203.100"
2020/03/03 13:01:24 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.197.35, server: , request: "GET / HTTP/1.1", upstream: "http://172.29.0.7:5601/", host: "10.20.203.100"
2020/03/03 13:01:24 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.197.35, server: , request: "GET / HTTP/1.1", upstream: "http://172.29.0.7:5601/", host: "10.20.203.100"
2020/03/03 13:01:25 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.197.35, server: , request: "GET / HTTP/1.1", upstream: "http://172.29.0.7:5601/", host: "10.20.203.100"
2020/03/03 13:01:25 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.197.35, server: , request: "GET / HTTP/1.1", upstream: "http://172.29.0.7:5601/", host: "10.20.203.100"
2020/03/03 13:01:25 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.197.35, server: , request: "GET / HTTP/1.1", upstream: "http://172.29.0.7:5601/", host: "10.20.203.100"
2020/03/03 13:01:32 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.197.35, server: , request: "GET / HTTP/1.1", upstream: "http://172.29.0.7:5601/", host: "10.20.203.100"
2020/03/03 13:01:33 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.197.35, server: , request: "GET / HTTP/1.1", upstream: "http://172.29.0.7:5601/", host: "10.20.203.100"
2020/03/03 13:01:33 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.197.35, server: , request: "GET / HTTP/1.1", upstream: "http://172.29.0.7:5601/", host: "10.20.203.100"
2020/03/03 13:01:33 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.197.35, server: , request: "GET / HTTP/1.1", upstream: "http://172.29.0.7:5601/", host: "10.20.203.100"
2020/03/03 13:01:33 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.197.35, server: , request: "GET / HTTP/1.1", upstream: "http://172.29.0.7:5601/", host: "10.20.203.100"
2020/03/03 13:01:34 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.197.35, server: , request: "GET / HTTP/1.1", upstream: "http://172.29.0.7:5601/", host: "10.20.203.100"
2020/03/03 13:01:35 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.197.35, server: , request: "GET / HTTP/1.1", upstream: "http://172.29.0.7:5601/", host: "10.20.203.100"
2020/03/03 13:01:36 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.197.35, server: , request: "GET / HTTP/1.1", upstream: "http://172.29.0.7:5601/", host: "10.20.203.100"
2020/03/03 13:01:38 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.197.35, server: , request: "GET / HTTP/1.1", upstream: "http://172.29.0.7:5601/", host: "10.20.203.100"
2020/03/03 13:01:44 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.197.35, server: , request: "GET / HTTP/1.1", upstream: "http://172.29.0.7:5601/", host: "10.20.203.100"
2020/03/03 13:01:48 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.197.35, server: , request: "GET / HTTP/1.1", upstream: "http://172.29.0.7:5601/", host: "10.20.203.100"
2020/03/03 13:01:54 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.197.35, server: , request: "GET / HTTP/1.1", upstream: "http://172.29.0.7:5601/", host: "10.20.203.100"
2020/03/03 13:01:54 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.197.35, server: , request: "GET / HTTP/1.1", upstream: "http://172.29.0.7:5601/", host: "10.20.203.100"
2020/03/03 13:01:55 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.197.35, server: , request: "GET / HTTP/1.1", upstream: "http://172.29.0.7:5601/", host: "10.20.203.100"
2020/03/03 13:01:56 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.197.35, server: , request: "GET / HTTP/1.1", upstream: "http://172.29.0.7:5601/", host: "10.20.203.100"
2020/03/03 13:01:57 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.197.35, server: , request: "GET / HTTP/1.1", upstream: "http://172.29.0.7:5601/", host: "10.20.203.100"
2020/03/03 13:01:57 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.197.35, server: , request: "GET / HTTP/1.1", upstream: "http://172.29.0.7:5601/", host: "10.20.203.100"
2020/03/03 13:01:58 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.197.35, server: , request: "GET / HTTP/1.1", upstream: "http://172.29.0.7:5601/", host: "10.20.203.100"
2020/03/03 13:01:58 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.197.35, server: , request: "GET / HTTP/1.1", upstream: "http://172.29.0.7:5601/", host: "10.20.203.100"
2020/03/03 13:01:58 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.197.35, server: , request: "GET / HTTP/1.1", upstream: "http://172.29.0.7:5601/", host: "10.20.203.100"
2020/03/03 13:01:58 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.197.35, server: , request: "GET / HTTP/1.1", upstream: "http://172.29.0.7:5601/", host: "10.20.203.100"
2020/03/03 13:01:59 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.197.35, server: , request: "GET / HTTP/1.1", upstream: "http://172.29.0.7:5601/", host: "10.20.203.100"
2020/03/03 13:01:59 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.197.35, server: , request: "GET / HTTP/1.1", upstream: "http://172.29.0.7:5601/", host: "10.20.203.100"
2020/03/03 13:02:00 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.197.35, server: , request: "GET / HTTP/1.1", upstream: "http://172.29.0.7:5601/", host: "10.20.203.100"
2020/03/03 13:02:00 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.197.35, server: , request: "GET / HTTP/1.1", upstream: "http://172.29.0.7:5601/", host: "10.20.203.100"
2020/03/03 13:02:00 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.197.35, server: , request: "GET / HTTP/1.1", upstream: "http://172.29.0.7:5601/", host: "10.20.203.100"
2020/03/03 13:02:01 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.197.35, server: , request: "GET / HTTP/1.1", upstream: "http://172.29.0.7:5601/", host: "10.20.203.100"
2020/03/03 13:02:01 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.197.35, server: , request: "GET / HTTP/1.1", upstream: "http://172.29.0.7:5601/", host: "10.20.203.100"
2020/03/03 13:02:07 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.197.35, server: , request: "GET / HTTP/1.1", upstream: "http://172.29.0.7:5601/", host: "10.20.203.100"
2020/03/03 13:02:08 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.197.35, server: , request: "GET / HTTP/1.1", upstream: "http://172.29.0.7:5601/", host: "10.20.203.100"
2020/03/03 13:02:09 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.197.35, server: , request: "GET / HTTP/1.1", upstream: "http://172.29.0.7:5601/", host: "10.20.203.100"
2020/03/03 13:02:09 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.197.35, server: , request: "GET / HTTP/1.1", upstream: "http://172.29.0.7:5601/", host: "10.20.203.100"
2020/03/03 13:02:09 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.197.35, server: , request: "GET / HTTP/1.1", upstream: "http://172.29.0.7:5601/", host: "10.20.203.100"
2020/03/03 13:02:21 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.197.35, server: , request: "GET / HTTP/1.1", upstream: "http://172.29.0.7:5601/", host: "10.20.203.100"
2020/03/03 13:02:28 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.197.35, server: , request: "GET / HTTP/1.1", upstream: "http://172.29.0.7:5601/", host: "10.20.203.100"
2020/03/03 13:02:38 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.197.35, server: , request: "GET / HTTP/1.1", upstream: "http://172.29.0.7:5601/", host: "10.20.203.100"
2020/03/03 13:02:43 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.197.35, server: , request: "GET / HTTP/1.1", upstream: "http://172.29.0.7:5601/", host: "10.20.203.100"
2020/03/03 13:03:33 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.197.35, server: , request: "GET / HTTP/1.1", upstream: "http://172.30.0.2:5601/", host: "10.20.203.100"
2020/03/03 13:03:34 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.197.35, server: , request: "GET / HTTP/1.1", upstream: "http://172.30.0.2:5601/", host: "10.20.203.100"
2020/03/03 13:03:34 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.197.35, server: , request: "GET / HTTP/1.1", upstream: "http://172.30.0.2:5601/", host: "10.20.203.100"
2020/03/03 13:03:34 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.197.35, server: , request: "GET / HTTP/1.1", upstream: "http://172.30.0.2:5601/", host: "10.20.203.100"
2020/03/03 13:03:34 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.197.35, server: , request: "GET / HTTP/1.1", upstream: "http://172.30.0.2:5601/", host: "10.20.203.100"
2020/03/03 13:03:35 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.197.35, server: , request: "GET / HTTP/1.1", upstream: "http://172.30.0.2:5601/", host: "10.20.203.100"
2020/03/03 13:03:36 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.197.35, server: , request: "GET / HTTP/1.1", upstream: "http://172.30.0.2:5601/", host: "10.20.203.100"
2020/03/03 13:03:36 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.197.35, server: , request: "GET / HTTP/1.1", upstream: "http://172.30.0.2:5601/", host: "10.20.203.100"
2020/03/03 13:03:36 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.197.35, server: , request: "GET / HTTP/1.1", upstream: "http://172.30.0.2:5601/", host: "10.20.203.100"
2020/03/03 13:03:37 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.197.35, server: , request: "GET / HTTP/1.1", upstream: "http://172.30.0.2:5601/", host: "10.20.203.100"
2020/03/03 13:03:37 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.197.35, server: , request: "GET / HTTP/1.1", upstream: "http://172.30.0.2:5601/", host: "10.20.203.100"
2020/03/03 13:03:37 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.197.35, server: , request: "GET / HTTP/1.1", upstream: "http://172.30.0.2:5601/", host: "10.20.203.100"
2020/03/03 13:03:38 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.197.35, server: , request: "GET / HTTP/1.1", upstream: "http://172.30.0.2:5601/", host: "10.20.203.100"
2020/03/03 13:03:38 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.197.35, server: , request: "GET / HTTP/1.1", upstream: "http://172.30.0.2:5601/", host: "10.20.203.100"
2020/03/03 13:03:38 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.197.35, server: , request: "GET / HTTP/1.1", upstream: "http://172.30.0.2:5601/", host: "10.20.203.100"
2020/03/03 13:03:39 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.197.35, server: , request: "GET / HTTP/1.1", upstream: "http://172.30.0.2:5601/", host: "10.20.203.100"
2020/03/03 13:03:39 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.197.35, server: , request: "GET / HTTP/1.1", upstream: "http://172.30.0.2:5601/", host: "10.20.203.100"
2020/03/03 13:03:39 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.197.35, server: , request: "GET / HTTP/1.1", upstream: "http://172.30.0.2:5601/", host: "10.20.203.100"
2020/03/03 13:03:39 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.197.35, server: , request: "GET / HTTP/1.1", upstream: "http://172.30.0.2:5601/", host: "10.20.203.100"
2020/03/03 13:03:39 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.197.35, server: , request: "GET / HTTP/1.1", upstream: "http://172.30.0.2:5601/", host: "10.20.203.100"
2020/03/03 13:03:39 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.197.35, server: , request: "GET / HTTP/1.1", upstream: "http://172.30.0.2:5601/", host: "10.20.203.100"
2020/03/03 13:03:39 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.197.35, server: , request: "GET / HTTP/1.1", upstream: "http://172.30.0.2:5601/", host: "10.20.203.100"
2020/03/03 13:03:40 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.197.35, server: , request: "GET / HTTP/1.1", upstream: "http://172.30.0.2:5601/", host: "10.20.203.100"
2020/03/03 13:03:40 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.197.35, server: , request: "GET / HTTP/1.1", upstream: "http://172.30.0.2:5601/", host: "10.20.203.100"
2020/03/03 13:03:46 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.197.35, server: , request: "GET / HTTP/1.1", upstream: "http://172.30.0.2:5601/", host: "10.20.203.100"
2020/03/03 13:03:47 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.197.35, server: , request: "GET / HTTP/1.1", upstream: "http://172.30.0.2:5601/", host: "10.20.203.100"
2020/03/03 13:03:48 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.197.35, server: , request: "GET / HTTP/1.1", upstream: "http://172.30.0.2:5601/", host: "10.20.203.100"
2020/03/03 13:03:48 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.197.35, server: , request: "GET / HTTP/1.1", upstream: "http://172.30.0.2:5601/", host: "10.20.203.100"
2020/03/03 13:03:48 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.197.35, server: , request: "GET / HTTP/1.1", upstream: "http://172.30.0.2:5601/", host: "10.20.203.100"
2020/03/03 13:06:19 [error] 6#6: *31 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.197.35, server: , request: "GET / HTTP/1.1", upstream: "http://172.30.0.2:5601/", host: "10.20.203.100"
2020/03/03 13:06:21 [error] 6#6: *31 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.197.35, server: , request: "GET / HTTP/1.1", upstream: "http://172.30.0.2:5601/", host: "10.20.203.100"
2020/03/03 13:06:21 [error] 6#6: *31 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.197.35, server: , request: "GET / HTTP/1.1", upstream: "http://172.30.0.2:5601/", host: "10.20.203.100"
2020/03/03 13:06:21 [error] 6#6: *31 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.197.35, server: , request: "GET / HTTP/1.1", upstream: "http://172.30.0.2:5601/", host: "10.20.203.100"
2020/03/03 13:06:21 [error] 6#6: *31 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.197.35, server: , request: "GET / HTTP/1.1", upstream: "http://172.30.0.2:5601/", host: "10.20.203.100"
2020/03/03 13:06:22 [error] 6#6: *31 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.197.35, server: , request: "GET / HTTP/1.1", upstream: "http://172.30.0.2:5601/", host: "10.20.203.100"
2020/03/03 13:06:22 [error] 6#6: *31 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.197.35, server: , request: "GET / HTTP/1.1", upstream: "http://172.30.0.2:5601/", host: "10.20.203.100"
2020/03/03 13:06:22 [error] 6#6: *31 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.197.35, server: , request: "GET / HTTP/1.1", upstream: "http://172.30.0.2:5601/", host: "10.20.203.100"
2020/03/03 13:06:22 [error] 6#6: *31 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.197.35, server: , request: "GET / HTTP/1.1", upstream: "http://172.30.0.2:5601/", host: "10.20.203.100"
2020/03/03 13:06:22 [error] 6#6: *31 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.197.35, server: , request: "GET / HTTP/1.1", upstream: "http://172.30.0.2:5601/", host: "10.20.203.100"
2020/03/03 13:06:22 [error] 6#6: *31 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.197.35, server: , request: "GET / HTTP/1.1", upstream: "http://172.30.0.2:5601/", host: "10.20.203.100"
2020/03/03 13:06:23 [error] 6#6: *31 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.197.35, server: , request: "GET / HTTP/1.1", upstream: "http://172.30.0.2:5601/", host: "10.20.203.100"
2020/03/03 13:06:24 [error] 6#6: *31 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.197.35, server: , request: "GET / HTTP/1.1", upstream: "http://172.30.0.2:5601/", host: "10.20.203.100"
2020/03/03 13:06:24 [error] 6#6: *31 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.197.35, server: , request: "GET / HTTP/1.1", upstream: "http://172.30.0.2:5601/", host: "10.20.203.100"
2020/03/03 13:06:24 [error] 6#6: *31 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.197.35, server: , request: "GET / HTTP/1.1", upstream: "http://172.30.0.2:5601/", host: "10.20.203.100"
2020/03/03 13:06:24 [error] 6#6: *31 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.197.35, server: , request: "GET / HTTP/1.1", upstream: "http://172.30.0.2:5601/", host: "10.20.203.100"
2020/03/03 13:06:34 [error] 6#6: *31 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.197.35, server: , request: "GET / HTTP/1.1", upstream: "http://172.30.0.2:5601/", host: "10.20.203.100"
2020/03/05 13:32:48 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.197.35, server: , request: "GET /app/kibana HTTP/1.1", upstream: "http://172.31.0.5:5601/app/kibana", host: "10.20.203.100"
2020/03/06 14:45:48 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.197.35, server: , request: "GET /app/kibana HTTP/1.1", upstream: "http://192.168.16.7:5601/app/kibana", host: "10.20.203.100"
2020/03/06 14:45:49 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.197.35, server: , request: "GET /app/kibana HTTP/1.1", upstream: "http://192.168.16.7:5601/app/kibana", host: "10.20.203.100"
2020/03/06 14:45:50 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.197.35, server: , request: "GET /app/kibana HTTP/1.1", upstream: "http://192.168.16.7:5601/app/kibana", host: "10.20.203.100"
2020/03/06 14:45:50 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.197.35, server: , request: "GET /app/kibana HTTP/1.1", upstream: "http://192.168.16.7:5601/app/kibana", host: "10.20.203.100"
2020/03/06 14:45:51 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.197.35, server: , request: "GET /app/kibana HTTP/1.1", upstream: "http://192.168.16.7:5601/app/kibana", host: "10.20.203.100"
2020/03/06 14:45:52 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.197.35, server: , request: "GET /app/kibana HTTP/1.1", upstream: "http://192.168.16.7:5601/app/kibana", host: "10.20.203.100"
2020/03/06 14:45:54 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.197.35, server: , request: "GET /app/kibana HTTP/1.1", upstream: "http://192.168.16.7:5601/app/kibana", host: "10.20.203.100"
2020/03/06 14:45:55 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.197.35, server: , request: "GET /app/kibana HTTP/1.1", upstream: "http://192.168.16.7:5601/app/kibana", host: "10.20.203.100"
2020/03/06 14:45:56 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.197.35, server: , request: "GET /app/kibana HTTP/1.1", upstream: "http://192.168.16.7:5601/app/kibana", host: "10.20.203.100"
2020/03/06 14:45:57 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.197.35, server: , request: "GET /app/kibana HTTP/1.1", upstream: "http://192.168.16.7:5601/app/kibana", host: "10.20.203.100"
2020/03/06 14:45:57 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.197.35, server: , request: "GET /app/kibana HTTP/1.1", upstream: "http://192.168.16.7:5601/app/kibana", host: "10.20.203.100"
2020/03/06 14:45:58 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.197.35, server: , request: "GET /app/kibana HTTP/1.1", upstream: "http://192.168.16.7:5601/app/kibana", host: "10.20.203.100"
2020/03/09 14:28:04 [emerg] 1#1: host not found in upstream "kib01:5601" in /etc/nginx/nginx.conf:8
2020/03/09 14:33:54 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.200.68, server: , request: "GET /app/kibana HTTP/1.1", upstream: "http://192.168.48.7:5601/app/kibana", host: "lamas.int"
2020/03/09 14:35:47 [error] 6#6: *7 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.200.68, server: , request: "GET / HTTP/1.1", upstream: "http://192.168.48.7:5601/", host: "lamas.int"
2020/03/09 14:47:23 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.200.68, server: , request: "GET / HTTP/1.1", upstream: "http://192.168.64.6:5601/", host: "lamas.int"
2020/03/09 14:47:24 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.200.68, server: , request: "GET / HTTP/1.1", upstream: "http://192.168.64.6:5601/", host: "lamas.int"
2020/03/09 14:47:25 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.200.68, server: , request: "GET / HTTP/1.1", upstream: "http://192.168.64.6:5601/", host: "lamas.int"
2020/03/09 14:47:25 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.200.68, server: , request: "GET / HTTP/1.1", upstream: "http://192.168.64.6:5601/", host: "lamas.int"
2020/03/09 14:47:25 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.200.68, server: , request: "GET / HTTP/1.1", upstream: "http://192.168.64.6:5601/", host: "lamas.int"
2020/03/09 14:47:26 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.200.68, server: , request: "GET / HTTP/1.1", upstream: "http://192.168.64.6:5601/", host: "lamas.int"
2020/03/09 14:47:26 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.200.68, server: , request: "GET / HTTP/1.1", upstream: "http://192.168.64.6:5601/", host: "lamas.int"
2020/03/09 14:47:26 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.200.68, server: , request: "GET / HTTP/1.1", upstream: "http://192.168.64.6:5601/", host: "lamas.int"
2020/03/09 14:47:26 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.200.68, server: , request: "GET / HTTP/1.1", upstream: "http://192.168.64.6:5601/", host: "lamas.int"
2020/03/09 14:47:27 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.200.68, server: , request: "GET / HTTP/1.1", upstream: "http://192.168.64.6:5601/", host: "lamas.int"
2020/03/09 14:47:28 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.200.68, server: , request: "GET / HTTP/1.1", upstream: "http://192.168.64.6:5601/", host: "lamas.int"
2020/03/09 14:47:28 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.200.68, server: , request: "GET / HTTP/1.1", upstream: "http://192.168.64.6:5601/", host: "lamas.int"
2020/03/09 14:47:28 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.200.68, server: , request: "GET / HTTP/1.1", upstream: "http://192.168.64.6:5601/", host: "lamas.int"
2020/03/09 14:47:28 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.200.68, server: , request: "GET / HTTP/1.1", upstream: "http://192.168.64.6:5601/", host: "lamas.int"
2020/03/09 14:47:28 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.200.68, server: , request: "GET / HTTP/1.1", upstream: "http://192.168.64.6:5601/", host: "lamas.int"
2020/03/09 14:47:29 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.200.68, server: , request: "GET / HTTP/1.1", upstream: "http://192.168.64.6:5601/", host: "lamas.int"
2020/03/09 14:47:31 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.200.68, server: , request: "GET / HTTP/1.1", upstream: "http://192.168.64.6:5601/", host: "lamas.int"
2020/03/09 14:47:33 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.200.68, server: , request: "GET / HTTP/1.1", upstream: "http://192.168.64.6:5601/", host: "lamas.int"
2020/03/09 14:47:33 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.200.68, server: , request: "GET / HTTP/1.1", upstream: "http://192.168.64.6:5601/", host: "lamas.int"
2020/03/09 14:47:37 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.200.68, server: , request: "GET / HTTP/1.1", upstream: "http://192.168.64.6:5601/", host: "lamas.int"
2020/03/09 14:47:57 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.200.68, server: , request: "GET / HTTP/1.1", upstream: "http://192.168.64.6:5601/", host: "lamas.int"
2020/03/09 14:47:58 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.200.68, server: , request: "GET / HTTP/1.1", upstream: "http://192.168.64.6:5601/", host: "lamas.int"
2020/03/09 14:47:59 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.200.68, server: , request: "GET / HTTP/1.1", upstream: "http://192.168.64.6:5601/", host: "lamas.int"
2020/03/09 14:47:59 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.200.68, server: , request: "GET / HTTP/1.1", upstream: "http://192.168.64.6:5601/", host: "lamas.int"
2020/03/09 14:47:59 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.200.68, server: , request: "GET / HTTP/1.1", upstream: "http://192.168.64.6:5601/", host: "lamas.int"
2020/03/09 14:47:59 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.200.68, server: , request: "GET / HTTP/1.1", upstream: "http://192.168.64.6:5601/", host: "lamas.int"
2020/03/09 14:47:59 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.200.68, server: , request: "GET / HTTP/1.1", upstream: "http://192.168.64.6:5601/", host: "lamas.int"
2020/03/09 14:48:00 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.200.68, server: , request: "GET / HTTP/1.1", upstream: "http://192.168.64.6:5601/", host: "lamas.int"
2020/03/09 14:48:00 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.200.68, server: , request: "GET / HTTP/1.1", upstream: "http://192.168.64.6:5601/", host: "lamas.int"
2020/03/09 14:48:00 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.200.68, server: , request: "GET / HTTP/1.1", upstream: "http://192.168.64.6:5601/", host: "lamas.int"
2020/03/09 14:49:31 [error] 6#6: *32 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.200.68, server: , request: "GET / HTTP/1.1", upstream: "http://192.168.64.6:5601/", host: "lamas.int"
2020/03/09 14:49:32 [error] 6#6: *32 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.200.68, server: , request: "GET / HTTP/1.1", upstream: "http://192.168.64.6:5601/", host: "lamas.int"
2020/03/09 14:49:33 [error] 6#6: *32 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.200.68, server: , request: "GET / HTTP/1.1", upstream: "http://192.168.64.6:5601/", host: "lamas.int"
2020/03/09 14:49:34 [error] 6#6: *32 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.200.68, server: , request: "GET / HTTP/1.1", upstream: "http://192.168.64.6:5601/", host: "lamas.int"
2020/03/09 14:49:34 [error] 6#6: *32 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.200.68, server: , request: "GET / HTTP/1.1", upstream: "http://192.168.64.6:5601/", host: "lamas.int"
2020/03/09 14:49:34 [error] 6#6: *32 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.200.68, server: , request: "GET / HTTP/1.1", upstream: "http://192.168.64.6:5601/", host: "lamas.int"
2020/03/09 14:49:34 [error] 6#6: *32 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.200.68, server: , request: "GET / HTTP/1.1", upstream: "http://192.168.64.6:5601/", host: "lamas.int"
2020/03/09 14:49:34 [error] 6#6: *32 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.200.68, server: , request: "GET / HTTP/1.1", upstream: "http://192.168.64.6:5601/", host: "lamas.int"
2020/03/09 14:49:35 [error] 6#6: *32 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.200.68, server: , request: "GET / HTTP/1.1", upstream: "http://192.168.64.6:5601/", host: "lamas.int"
2020/03/09 14:49:35 [error] 6#6: *32 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.200.68, server: , request: "GET / HTTP/1.1", upstream: "http://192.168.64.6:5601/", host: "lamas.int"
2020/03/09 14:49:35 [error] 6#6: *32 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.200.68, server: , request: "GET / HTTP/1.1", upstream: "http://192.168.64.6:5601/", host: "lamas.int"
2020/03/09 14:50:48 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.200.68, server: , request: "GET / HTTP/1.1", upstream: "http://192.168.80.6:5601/", host: "lamas.int"
2020/03/09 14:50:48 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.200.68, server: , request: "GET / HTTP/1.1", upstream: "http://192.168.80.6:5601/", host: "lamas.int"
2020/03/09 14:50:48 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.200.68, server: , request: "GET / HTTP/1.1", upstream: "http://192.168.80.6:5601/", host: "lamas.int"
2020/03/09 14:50:48 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.200.68, server: , request: "GET / HTTP/1.1", upstream: "http://192.168.80.6:5601/", host: "lamas.int"
2020/03/09 14:50:49 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.200.68, server: , request: "GET / HTTP/1.1", upstream: "http://192.168.80.6:5601/", host: "lamas.int"
2020/03/09 14:50:49 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.200.68, server: , request: "GET / HTTP/1.1", upstream: "http://192.168.80.6:5601/", host: "lamas.int"
2020/03/09 14:50:49 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.200.68, server: , request: "GET / HTTP/1.1", upstream: "http://192.168.80.6:5601/", host: "lamas.int"
2020/03/09 14:50:49 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.200.68, server: , request: "GET / HTTP/1.1", upstream: "http://192.168.80.6:5601/", host: "lamas.int"
2020/03/09 14:50:50 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.200.68, server: , request: "GET / HTTP/1.1", upstream: "http://192.168.80.6:5601/", host: "lamas.int"
2020/03/09 14:50:50 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.200.68, server: , request: "GET / HTTP/1.1", upstream: "http://192.168.80.6:5601/", host: "lamas.int"
2020/03/09 14:50:51 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.200.68, server: , request: "GET / HTTP/1.1", upstream: "http://192.168.80.6:5601/", host: "lamas.int"
2020/03/09 14:50:51 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.200.68, server: , request: "GET / HTTP/1.1", upstream: "http://192.168.80.6:5601/", host: "lamas.int"
2020/03/09 14:50:52 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.200.68, server: , request: "GET / HTTP/1.1", upstream: "http://192.168.80.6:5601/", host: "lamas.int"
2020/03/09 14:50:52 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.200.68, server: , request: "GET / HTTP/1.1", upstream: "http://192.168.80.6:5601/", host: "lamas.int"
2020/03/09 14:50:52 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.200.68, server: , request: "GET / HTTP/1.1", upstream: "http://192.168.80.6:5601/", host: "lamas.int"
2020/03/09 14:50:52 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.200.68, server: , request: "GET / HTTP/1.1", upstream: "http://192.168.80.6:5601/", host: "lamas.int"
2020/03/09 14:50:53 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.200.68, server: , request: "GET / HTTP/1.1", upstream: "http://192.168.80.6:5601/", host: "lamas.int"
2020/03/09 14:50:53 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.200.68, server: , request: "GET / HTTP/1.1", upstream: "http://192.168.80.6:5601/", host: "lamas.int"
2020/03/09 14:50:53 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.200.68, server: , request: "GET / HTTP/1.1", upstream: "http://192.168.80.6:5601/", host: "lamas.int"
2020/03/09 14:50:53 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.200.68, server: , request: "GET / HTTP/1.1", upstream: "http://192.168.80.6:5601/", host: "lamas.int"
2020/03/09 14:50:54 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.200.68, server: , request: "GET / HTTP/1.1", upstream: "http://192.168.80.6:5601/", host: "lamas.int"
2020/03/09 14:50:54 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.200.68, server: , request: "GET / HTTP/1.1", upstream: "http://192.168.80.6:5601/", host: "lamas.int"
2020/03/09 14:50:54 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.200.68, server: , request: "GET / HTTP/1.1", upstream: "http://192.168.80.6:5601/", host: "lamas.int"
2020/03/09 14:50:54 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.200.68, server: , request: "GET / HTTP/1.1", upstream: "http://192.168.80.6:5601/", host: "lamas.int"
2020/03/09 14:50:54 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.200.68, server: , request: "GET / HTTP/1.1", upstream: "http://192.168.80.6:5601/", host: "lamas.int"
2020/03/09 14:50:55 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.200.68, server: , request: "GET / HTTP/1.1", upstream: "http://192.168.80.6:5601/", host: "lamas.int"
2020/03/09 14:50:55 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.200.68, server: , request: "GET / HTTP/1.1", upstream: "http://192.168.80.6:5601/", host: "lamas.int"
2020/03/09 14:50:55 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.200.68, server: , request: "GET / HTTP/1.1", upstream: "http://192.168.80.6:5601/", host: "lamas.int"
2020/03/09 14:50:56 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.200.68, server: , request: "GET / HTTP/1.1", upstream: "http://192.168.80.6:5601/", host: "lamas.int"
2020/03/09 14:50:56 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.200.68, server: , request: "GET / HTTP/1.1", upstream: "http://192.168.80.6:5601/", host: "lamas.int"
2020/03/09 14:50:56 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.200.68, server: , request: "GET / HTTP/1.1", upstream: "http://192.168.80.6:5601/", host: "lamas.int"
2020/03/09 14:50:56 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.200.68, server: , request: "GET / HTTP/1.1", upstream: "http://192.168.80.6:5601/", host: "lamas.int"
2020/03/09 14:50:57 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.200.68, server: , request: "GET / HTTP/1.1", upstream: "http://192.168.80.6:5601/", host: "lamas.int"
2020/03/09 14:50:57 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.200.68, server: , request: "GET / HTTP/1.1", upstream: "http://192.168.80.6:5601/", host: "lamas.int"
2020/03/09 14:50:57 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.200.68, server: , request: "GET / HTTP/1.1", upstream: "http://192.168.80.6:5601/", host: "lamas.int"
2020/03/09 14:50:57 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.200.68, server: , request: "GET / HTTP/1.1", upstream: "http://192.168.80.6:5601/", host: "lamas.int"
2020/03/09 14:50:58 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.200.68, server: , request: "GET / HTTP/1.1", upstream: "http://192.168.80.6:5601/", host: "lamas.int"
2020/03/09 14:50:58 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.200.68, server: , request: "GET / HTTP/1.1", upstream: "http://192.168.80.6:5601/", host: "lamas.int"
2020/03/09 14:50:58 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.200.68, server: , request: "GET / HTTP/1.1", upstream: "http://192.168.80.6:5601/", host: "lamas.int"
2020/03/09 14:50:59 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.200.68, server: , request: "GET / HTTP/1.1", upstream: "http://192.168.80.6:5601/", host: "lamas.int"
2020/03/09 14:50:59 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.200.68, server: , request: "GET / HTTP/1.1", upstream: "http://192.168.80.6:5601/", host: "lamas.int"
2020/03/09 14:50:59 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.200.68, server: , request: "GET / HTTP/1.1", upstream: "http://192.168.80.6:5601/", host: "lamas.int"
2020/03/09 14:50:59 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.200.68, server: , request: "GET / HTTP/1.1", upstream: "http://192.168.80.6:5601/", host: "lamas.int"
2020/03/09 14:50:59 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.200.68, server: , request: "GET / HTTP/1.1", upstream: "http://192.168.80.6:5601/", host: "lamas.int"
2020/03/09 14:51:00 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.200.68, server: , request: "GET / HTTP/1.1", upstream: "http://192.168.80.6:5601/", host: "lamas.int"
2020/03/09 14:51:00 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.200.68, server: , request: "GET / HTTP/1.1", upstream: "http://192.168.80.6:5601/", host: "lamas.int"
2020/03/09 14:51:00 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.200.68, server: , request: "GET / HTTP/1.1", upstream: "http://192.168.80.6:5601/", host: "lamas.int"
2020/03/09 14:51:00 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.200.68, server: , request: "GET / HTTP/1.1", upstream: "http://192.168.80.6:5601/", host: "lamas.int"
2020/03/09 14:51:00 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.200.68, server: , request: "GET / HTTP/1.1", upstream: "http://192.168.80.6:5601/", host: "lamas.int"
2020/03/09 14:51:00 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.200.68, server: , request: "GET / HTTP/1.1", upstream: "http://192.168.80.6:5601/", host: "lamas.int"
2020/03/09 14:51:01 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.200.68, server: , request: "GET / HTTP/1.1", upstream: "http://192.168.80.6:5601/", host: "lamas.int"
2020/03/09 14:51:01 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.200.68, server: , request: "GET / HTTP/1.1", upstream: "http://192.168.80.6:5601/", host: "lamas.int"
2020/03/09 14:51:01 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.200.68, server: , request: "GET / HTTP/1.1", upstream: "http://192.168.80.6:5601/", host: "lamas.int"
2020/03/09 14:51:01 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.200.68, server: , request: "GET / HTTP/1.1", upstream: "http://192.168.80.6:5601/", host: "lamas.int"
2020/03/09 14:51:01 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.200.68, server: , request: "GET / HTTP/1.1", upstream: "http://192.168.80.6:5601/", host: "lamas.int"
2020/03/09 14:51:02 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.200.68, server: , request: "GET / HTTP/1.1", upstream: "http://192.168.80.6:5601/", host: "lamas.int"
2020/03/09 14:51:02 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.200.68, server: , request: "GET / HTTP/1.1", upstream: "http://192.168.80.6:5601/", host: "lamas.int"
2020/03/09 14:51:03 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.200.68, server: , request: "GET / HTTP/1.1", upstream: "http://192.168.80.6:5601/", host: "lamas.int"
2020/03/09 14:51:03 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.200.68, server: , request: "GET / HTTP/1.1", upstream: "http://192.168.80.6:5601/", host: "lamas.int"
2020/03/09 14:51:03 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.200.68, server: , request: "GET / HTTP/1.1", upstream: "http://192.168.80.6:5601/", host: "lamas.int"
2020/03/09 14:51:10 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.200.68, server: , request: "GET / HTTP/1.1", upstream: "http://192.168.80.6:5601/", host: "lamas.int"
2020/03/09 14:51:11 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.200.68, server: , request: "GET / HTTP/1.1", upstream: "http://192.168.80.6:5601/", host: "lamas.int"
2020/03/09 14:51:11 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.200.68, server: , request: "GET / HTTP/1.1", upstream: "http://192.168.80.6:5601/", host: "lamas.int"
2020/03/09 14:51:11 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.200.68, server: , request: "GET / HTTP/1.1", upstream: "http://192.168.80.6:5601/", host: "lamas.int"
2020/03/09 15:09:56 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.200.68, server: , request: "GET / HTTP/1.1", upstream: "http://192.168.96.2:5601/", host: "lamas.int"
2020/03/09 15:09:56 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.200.68, server: , request: "GET / HTTP/1.1", upstream: "http://192.168.96.2:5601/", host: "lamas.int"
2020/03/09 15:09:56 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.200.68, server: , request: "GET / HTTP/1.1", upstream: "http://192.168.96.2:5601/", host: "lamas.int"
2020/03/09 15:09:56 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.200.68, server: , request: "GET / HTTP/1.1", upstream: "http://192.168.96.2:5601/", host: "lamas.int"
2020/03/09 15:09:56 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.200.68, server: , request: "GET / HTTP/1.1", upstream: "http://192.168.96.2:5601/", host: "lamas.int"
2020/03/09 15:11:17 [error] 6#6: *11 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.200.68, server: , request: "GET / HTTP/1.1", upstream: "http://192.168.96.2:5601/", host: "lamas.int"
2020/03/09 15:11:18 [error] 6#6: *11 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.200.68, server: , request: "GET / HTTP/1.1", upstream: "http://192.168.96.2:5601/", host: "lamas.int"
2020/03/09 15:11:18 [error] 6#6: *11 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.200.68, server: , request: "GET / HTTP/1.1", upstream: "http://192.168.96.2:5601/", host: "lamas.int"
2020/03/09 15:11:18 [error] 6#6: *11 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.200.68, server: , request: "GET / HTTP/1.1", upstream: "http://192.168.96.2:5601/", host: "lamas.int"
2020/03/09 15:11:18 [error] 6#6: *11 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.200.68, server: , request: "GET / HTTP/1.1", upstream: "http://192.168.96.2:5601/", host: "lamas.int"
2020/03/09 15:11:18 [error] 6#6: *11 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.200.68, server: , request: "GET / HTTP/1.1", upstream: "http://192.168.96.2:5601/", host: "lamas.int"
2020/03/09 15:11:19 [error] 6#6: *11 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.200.68, server: , request: "GET / HTTP/1.1", upstream: "http://192.168.96.2:5601/", host: "lamas.int"
2020/03/09 15:11:19 [error] 6#6: *11 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.200.68, server: , request: "GET / HTTP/1.1", upstream: "http://192.168.96.2:5601/", host: "lamas.int"
2020/03/09 15:11:19 [error] 6#6: *11 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.200.68, server: , request: "GET / HTTP/1.1", upstream: "http://192.168.96.2:5601/", host: "lamas.int"
2020/03/09 15:13:02 [error] 6#6: *21 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.200.68, server: , request: "GET / HTTP/1.1", upstream: "http://192.168.96.2:5601/", host: "lamas.int"
2020/03/09 15:13:03 [error] 6#6: *21 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.200.68, server: , request: "GET / HTTP/1.1", upstream: "http://192.168.96.2:5601/", host: "lamas.int"
2020/03/09 15:13:04 [error] 6#6: *21 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.200.68, server: , request: "GET / HTTP/1.1", upstream: "http://192.168.96.2:5601/", host: "lamas.int"
2020/03/09 15:13:04 [error] 6#6: *21 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.200.68, server: , request: "GET / HTTP/1.1", upstream: "http://192.168.96.2:5601/", host: "lamas.int"
2020/03/09 15:13:04 [error] 6#6: *21 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.200.68, server: , request: "GET / HTTP/1.1", upstream: "http://192.168.96.2:5601/", host: "lamas.int"
2020/03/09 15:13:04 [error] 6#6: *21 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.200.68, server: , request: "GET / HTTP/1.1", upstream: "http://192.168.96.2:5601/", host: "lamas.int"
2020/03/09 15:13:05 [error] 6#6: *21 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.200.68, server: , request: "GET / HTTP/1.1", upstream: "http://192.168.96.2:5601/", host: "lamas.int"
2020/03/09 15:13:05 [error] 6#6: *21 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.200.68, server: , request: "GET / HTTP/1.1", upstream: "http://192.168.96.2:5601/", host: "lamas.int"
2020/03/09 15:13:05 [error] 6#6: *21 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.200.68, server: , request: "GET / HTTP/1.1", upstream: "http://192.168.96.2:5601/", host: "lamas.int"
2020/03/09 15:13:05 [error] 6#6: *21 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.200.68, server: , request: "GET / HTTP/1.1", upstream: "http://192.168.96.2:5601/", host: "lamas.int"
2020/03/09 15:13:05 [error] 6#6: *21 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.200.68, server: , request: "GET / HTTP/1.1", upstream: "http://192.168.96.2:5601/", host: "lamas.int"
2020/03/09 15:13:06 [error] 6#6: *21 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.200.68, server: , request: "GET / HTTP/1.1", upstream: "http://192.168.96.2:5601/", host: "lamas.int"
2020/03/09 15:13:06 [error] 6#6: *21 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.200.68, server: , request: "GET / HTTP/1.1", upstream: "http://192.168.96.2:5601/", host: "lamas.int"
2020/03/09 15:13:06 [error] 6#6: *21 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.200.68, server: , request: "GET / HTTP/1.1", upstream: "http://192.168.96.2:5601/", host: "lamas.int"
2020/03/09 15:13:06 [error] 6#6: *21 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.200.68, server: , request: "GET / HTTP/1.1", upstream: "http://192.168.96.2:5601/", host: "lamas.int"
2020/03/09 15:13:07 [error] 6#6: *21 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.200.68, server: , request: "GET / HTTP/1.1", upstream: "http://192.168.96.2:5601/", host: "lamas.int"
2020/03/09 15:13:07 [error] 6#6: *21 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.200.68, server: , request: "GET / HTTP/1.1", upstream: "http://192.168.96.2:5601/", host: "lamas.int"
2020/03/09 15:13:08 [error] 6#6: *21 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.200.68, server: , request: "GET / HTTP/1.1", upstream: "http://192.168.96.2:5601/", host: "lamas.int"
2020/03/09 15:13:08 [error] 6#6: *21 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.200.68, server: , request: "GET / HTTP/1.1", upstream: "http://192.168.96.2:5601/", host: "lamas.int"
2020/03/09 15:13:09 [error] 6#6: *21 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.200.68, server: , request: "GET / HTTP/1.1", upstream: "http://192.168.96.2:5601/", host: "lamas.int"
2020/03/09 15:13:10 [error] 6#6: *21 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.200.68, server: , request: "GET / HTTP/1.1", upstream: "http://192.168.96.2:5601/", host: "lamas.int"
2020/03/09 15:13:11 [error] 6#6: *21 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.200.68, server: , request: "GET / HTTP/1.1", upstream: "http://192.168.96.2:5601/", host: "lamas.int"
2020/03/09 15:13:11 [error] 6#6: *21 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.200.68, server: , request: "GET / HTTP/1.1", upstream: "http://192.168.96.2:5601/", host: "lamas.int"
2020/03/09 15:30:57 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.200.68, server: , request: "GET / HTTP/1.1", upstream: "http://192.168.112.6:5601/", host: "lamas.int"
2020/03/09 15:30:58 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.200.68, server: , request: "GET / HTTP/1.1", upstream: "http://192.168.112.6:5601/", host: "lamas.int"
2020/03/09 15:30:58 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.200.68, server: , request: "GET / HTTP/1.1", upstream: "http://192.168.112.6:5601/", host: "lamas.int"
2020/03/09 15:30:58 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.200.68, server: , request: "GET / HTTP/1.1", upstream: "http://192.168.112.6:5601/", host: "lamas.int"
2020/03/09 15:30:58 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.200.68, server: , request: "GET / HTTP/1.1", upstream: "http://192.168.112.6:5601/", host: "lamas.int"
2020/03/09 15:30:58 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.200.68, server: , request: "GET / HTTP/1.1", upstream: "http://192.168.112.6:5601/", host: "lamas.int"
2020/03/09 15:30:58 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.200.68, server: , request: "GET / HTTP/1.1", upstream: "http://192.168.112.6:5601/", host: "lamas.int"
2020/03/09 15:30:59 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.200.68, server: , request: "GET / HTTP/1.1", upstream: "http://192.168.112.6:5601/", host: "lamas.int"
2020/03/09 15:30:59 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.200.68, server: , request: "GET / HTTP/1.1", upstream: "http://192.168.112.6:5601/", host: "lamas.int"
2020/03/09 15:30:59 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.200.68, server: , request: "GET / HTTP/1.1", upstream: "http://192.168.112.6:5601/", host: "lamas.int"
2020/03/09 15:31:00 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.200.68, server: , request: "GET / HTTP/1.1", upstream: "http://192.168.112.6:5601/", host: "lamas.int"
2020/03/09 15:31:01 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.200.68, server: , request: "GET / HTTP/1.1", upstream: "http://192.168.112.6:5601/", host: "lamas.int"
2020/03/09 15:31:01 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.200.68, server: , request: "GET / HTTP/1.1", upstream: "http://192.168.112.6:5601/", host: "lamas.int"
2020/03/09 15:31:01 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.200.68, server: , request: "GET / HTTP/1.1", upstream: "http://192.168.112.6:5601/", host: "lamas.int"
2020/03/09 15:31:02 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.200.68, server: , request: "GET / HTTP/1.1", upstream: "http://192.168.112.6:5601/", host: "lamas.int"
2020/03/09 15:31:02 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.200.68, server: , request: "GET / HTTP/1.1", upstream: "http://192.168.112.6:5601/", host: "lamas.int"
2020/03/09 15:31:03 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.200.68, server: , request: "GET / HTTP/1.1", upstream: "http://192.168.112.6:5601/", host: "lamas.int"
2020/03/09 15:31:04 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.200.68, server: , request: "GET / HTTP/1.1", upstream: "http://192.168.112.6:5601/", host: "lamas.int"
2020/03/09 15:31:04 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.200.68, server: , request: "GET / HTTP/1.1", upstream: "http://192.168.112.6:5601/", host: "lamas.int"
2020/03/09 15:31:04 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.200.68, server: , request: "GET / HTTP/1.1", upstream: "http://192.168.112.6:5601/", host: "lamas.int"
2020/03/09 15:31:05 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.200.68, server: , request: "GET / HTTP/1.1", upstream: "http://192.168.112.6:5601/", host: "lamas.int"
2020/03/09 15:31:05 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.200.68, server: , request: "GET / HTTP/1.1", upstream: "http://192.168.112.6:5601/", host: "lamas.int"
2020/03/09 15:31:05 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.200.68, server: , request: "GET / HTTP/1.1", upstream: "http://192.168.112.6:5601/", host: "lamas.int"
2020/03/09 15:31:05 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.200.68, server: , request: "GET / HTTP/1.1", upstream: "http://192.168.112.6:5601/", host: "lamas.int"
2020/03/09 15:31:14 [error] 6#6: *26 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.200.68, server: , request: "GET / HTTP/1.1", upstream: "http://192.168.112.6:5601/", host: "10.20.203.100"
2020/03/09 15:31:19 [error] 6#6: *26 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.200.68, server: , request: "GET / HTTP/1.1", upstream: "http://192.168.112.6:5601/", host: "10.20.203.100"
2020/03/09 15:32:06 [error] 6#6: *30 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.200.68, server: , request: "GET / HTTP/1.1", upstream: "http://192.168.112.6:5601/", host: "lamas.int"
2020/03/09 15:32:07 [error] 6#6: *30 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.200.68, server: , request: "GET / HTTP/1.1", upstream: "http://192.168.112.6:5601/", host: "lamas.int"
2020/03/09 15:32:08 [error] 6#6: *30 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.200.68, server: , request: "GET / HTTP/1.1", upstream: "http://192.168.112.6:5601/", host: "lamas.int"
2020/03/09 15:32:08 [error] 6#6: *30 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.200.68, server: , request: "GET / HTTP/1.1", upstream: "http://192.168.112.6:5601/", host: "lamas.int"
2020/03/09 15:32:08 [error] 6#6: *30 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.200.68, server: , request: "GET / HTTP/1.1", upstream: "http://192.168.112.6:5601/", host: "lamas.int"
2020/03/09 15:32:08 [error] 6#6: *30 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.200.68, server: , request: "GET / HTTP/1.1", upstream: "http://192.168.112.6:5601/", host: "lamas.int"
2020/03/09 15:32:08 [error] 6#6: *30 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.200.68, server: , request: "GET / HTTP/1.1", upstream: "http://192.168.112.6:5601/", host: "lamas.int"
2020/03/09 15:32:08 [error] 6#6: *30 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.200.68, server: , request: "GET / HTTP/1.1", upstream: "http://192.168.112.6:5601/", host: "lamas.int"
2020/03/09 15:32:08 [error] 6#6: *30 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.200.68, server: , request: "GET / HTTP/1.1", upstream: "http://192.168.112.6:5601/", host: "lamas.int"
2020/03/09 15:32:09 [error] 6#6: *30 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.200.68, server: , request: "GET / HTTP/1.1", upstream: "http://192.168.112.6:5601/", host: "lamas.int"
2020/03/09 15:32:09 [error] 6#6: *30 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.200.68, server: , request: "GET / HTTP/1.1", upstream: "http://192.168.112.6:5601/", host: "lamas.int"
2020/03/09 15:32:09 [error] 6#6: *30 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.200.68, server: , request: "GET / HTTP/1.1", upstream: "http://192.168.112.6:5601/", host: "lamas.int"
2020/03/09 15:32:10 [error] 6#6: *30 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.200.68, server: , request: "GET / HTTP/1.1", upstream: "http://192.168.112.6:5601/", host: "lamas.int"
2020/03/09 15:32:10 [error] 6#6: *30 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.200.68, server: , request: "GET / HTTP/1.1", upstream: "http://192.168.112.6:5601/", host: "lamas.int"
2020/03/09 15:32:10 [error] 6#6: *30 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.200.68, server: , request: "GET / HTTP/1.1", upstream: "http://192.168.112.6:5601/", host: "lamas.int"
2020/03/09 15:32:10 [error] 6#6: *30 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.200.68, server: , request: "GET / HTTP/1.1", upstream: "http://192.168.112.6:5601/", host: "lamas.int"
2020/03/09 15:32:10 [error] 6#6: *30 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.200.68, server: , request: "GET / HTTP/1.1", upstream: "http://192.168.112.6:5601/", host: "lamas.int"
2020/03/09 15:32:17 [error] 6#6: *30 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.200.68, server: , request: "GET / HTTP/1.1", upstream: "http://192.168.112.6:5601/", host: "lamas.int"
2020/03/09 15:32:18 [error] 6#6: *30 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.200.68, server: , request: "GET / HTTP/1.1", upstream: "http://192.168.112.6:5601/", host: "lamas.int"
2020/03/09 15:32:18 [error] 6#6: *30 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.200.68, server: , request: "GET / HTTP/1.1", upstream: "http://192.168.112.6:5601/", host: "lamas.int"
2020/03/09 15:32:18 [error] 6#6: *30 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.200.68, server: , request: "GET / HTTP/1.1", upstream: "http://192.168.112.6:5601/", host: "lamas.int"
2020/03/09 15:32:19 [error] 6#6: *30 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.200.68, server: , request: "GET / HTTP/1.1", upstream: "http://192.168.112.6:5601/", host: "lamas.int"
2020/03/09 15:32:19 [error] 6#6: *30 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.200.68, server: , request: "GET / HTTP/1.1", upstream: "http://192.168.112.6:5601/", host: "lamas.int"
2020/03/09 15:32:19 [error] 6#6: *30 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.200.68, server: , request: "GET / HTTP/1.1", upstream: "http://192.168.112.6:5601/", host: "lamas.int"
2020/03/09 15:32:19 [error] 6#6: *30 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.200.68, server: , request: "GET / HTTP/1.1", upstream: "http://192.168.112.6:5601/", host: "lamas.int"
2020/03/10 13:07:00 [error] 6#6: *1 connect() failed (113: No route to host) while connecting to upstream, client: 10.20.197.35, server: , request: "GET /app/kibana HTTP/1.1", upstream: "http://192.168.112.6:5601/app/kibana", host: "10.20.203.100"
2020/03/10 13:10:17 [error] 6#6: *9 connect() failed (113: No route to host) while connecting to upstream, client: 10.20.197.35, server: , request: "GET /app/kibana HTTP/1.1", upstream: "http://192.168.112.6:5601/app/kibana", host: "10.20.203.100"
2020/03/10 13:33:25 [emerg] 1#1: host not found in upstream "kib01:5601" in /etc/nginx/nginx.conf:8
2020/03/10 14:23:56 [emerg] 1#1: host not found in upstream "kibana:5601" in /etc/nginx/nginx.conf:8
2020/03/12 12:50:40 [emerg] 1#1: host not found in upstream "kibana:5601" in /etc/nginx/nginx.conf:8
2020/03/12 12:52:32 [emerg] 1#1: host not found in upstream "kibana" in /etc/nginx/nginx.conf:8
2020/03/12 12:58:03 [emerg] 1#1: host not found in upstream "kibana" in /etc/nginx/nginx.conf:8
2020/03/12 12:59:26 [emerg] 1#1: host not found in upstream "kibana:5601" in /etc/nginx/nginx.conf:8
2020/03/12 13:01:27 [emerg] 1#1: host not found in upstream "kibana:5601" in /etc/nginx/nginx.conf:8
2020/03/12 13:06:57 [emerg] 1#1: host not found in upstream "kibana:5601" in /etc/nginx/nginx.conf:8
2020/03/12 13:12:51 [emerg] 1#1: host not found in upstream "kibana" in /etc/nginx/nginx.conf:19
2020/03/12 13:16:09 [emerg] 1#1: host not found in upstream "kibana:5601" in /etc/nginx/nginx.conf:8
2020/03/12 13:17:38 [error] 6#6: *1 recv() failed (104: Connection reset by peer) while reading response header from upstream, client: 10.20.197.35, server: , request: "GET /app/kibana HTTP/1.1", upstream: "http://10.20.203.100:5601/app/kibana", host: "10.20.203.100"
2020/03/12 13:26:07 [error] 6#6: *2 upstream prematurely closed connection while reading response header from upstream, client: 10.20.197.35, server: , request: "GET / HTTP/1.1", upstream: "http://10.20.203.100:5601/", host: "10.20.203.100"
2020/03/12 13:30:07 [error] 6#6: *1 recv() failed (104: Connection reset by peer) while reading response header from upstream, client: 10.20.197.35, server: , request: "GET / HTTP/1.1", upstream: "http://10.20.203.100:5601/", host: "10.20.203.100"
2020/03/12 13:31:05 [error] 6#6: *1 upstream prematurely closed connection while reading response header from upstream, client: 10.20.197.35, server: , request: "GET / HTTP/1.1", upstream: "http://10.20.203.100:5601/", host: "10.20.203.100"
2020/03/12 13:31:55 [error] 6#6: *1 upstream prematurely closed connection while reading response header from upstream, client: 10.20.197.35, server: , request: "GET / HTTP/1.1", upstream: "http://10.20.203.100:5601/", host: "10.20.203.100"
2020/03/12 14:27:49 [emerg] 1#1: host not found in upstream "kibana:5601" in /etc/nginx/nginx.conf:8
2020/03/12 14:54:18 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.197.35, server: , request: "GET / HTTP/1.1", upstream: "http://127.0.0.1:5601/", host: "10.20.203.100"
2020/03/12 14:54:18 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.197.35, server: , request: "GET / HTTP/1.1", upstream: "http://127.0.0.1:5601/", host: "10.20.203.100"
2020/03/12 14:54:19 [error] 6#6: *1 no live upstreams while connecting to upstream, client: 10.20.197.35, server: , request: "GET / HTTP/1.1", upstream: "http://docker-kibana/", host: "10.20.203.100"
2020/03/12 15:02:34 [emerg] 1#1: host not found in upstream "kib01:5601" in /etc/nginx/nginx.conf:8
2020/03/12 15:04:38 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.197.35, server: , request: "GET / HTTP/1.1", upstream: "http://127.0.0.1:5601/", host: "10.20.203.100"
2020/03/12 15:04:38 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.197.35, server: , request: "GET / HTTP/1.1", upstream: "http://127.0.0.1:5601/", host: "10.20.203.100"
2020/03/12 15:04:39 [error] 6#6: *1 no live upstreams while connecting to upstream, client: 10.20.197.35, server: , request: "GET / HTTP/1.1", upstream: "http://docker-kibana/", host: "10.20.203.100"
2020/03/12 15:05:17 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.197.35, server: , request: "GET / HTTP/1.1", upstream: "http://127.0.0.1:5601/", host: "10.20.203.100"
2020/03/12 15:05:17 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.197.35, server: , request: "GET / HTTP/1.1", upstream: "http://127.0.0.1:5601/", host: "10.20.203.100"
2020/03/12 15:05:19 [error] 6#6: *1 no live upstreams while connecting to upstream, client: 10.20.197.35, server: , request: "GET / HTTP/1.1", upstream: "http://docker-kibana/", host: "10.20.203.100"
2020/03/12 15:05:19 [error] 6#6: *1 no live upstreams while connecting to upstream, client: 10.20.197.35, server: , request: "GET / HTTP/1.1", upstream: "http://docker-kibana/", host: "10.20.203.100"
2020/03/12 15:05:19 [error] 6#6: *1 no live upstreams while connecting to upstream, client: 10.20.197.35, server: , request: "GET / HTTP/1.1", upstream: "http://docker-kibana/", host: "10.20.203.100"
2020/03/12 15:05:19 [error] 6#6: *1 no live upstreams while connecting to upstream, client: 10.20.197.35, server: , request: "GET / HTTP/1.1", upstream: "http://docker-kibana/", host: "10.20.203.100"
2020/03/12 15:05:19 [error] 6#6: *1 no live upstreams while connecting to upstream, client: 10.20.197.35, server: , request: "GET / HTTP/1.1", upstream: "http://docker-kibana/", host: "10.20.203.100"
2020/03/12 15:05:19 [error] 6#6: *1 no live upstreams while connecting to upstream, client: 10.20.197.35, server: , request: "GET / HTTP/1.1", upstream: "http://docker-kibana/", host: "10.20.203.100"
2020/03/12 15:05:19 [error] 6#6: *1 no live upstreams while connecting to upstream, client: 10.20.197.35, server: , request: "GET / HTTP/1.1", upstream: "http://docker-kibana/", host: "10.20.203.100"
2020/03/12 15:05:20 [error] 6#6: *1 no live upstreams while connecting to upstream, client: 10.20.197.35, server: , request: "GET / HTTP/1.1", upstream: "http://docker-kibana/", host: "10.20.203.100"
2020/03/12 15:05:20 [error] 6#6: *1 no live upstreams while connecting to upstream, client: 10.20.197.35, server: , request: "GET / HTTP/1.1", upstream: "http://docker-kibana/", host: "10.20.203.100"
2020/03/12 15:05:20 [error] 6#6: *1 no live upstreams while connecting to upstream, client: 10.20.197.35, server: , request: "GET / HTTP/1.1", upstream: "http://docker-kibana/", host: "10.20.203.100"
2020/03/12 15:06:07 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.197.35, server: , request: "GET / HTTP/1.1", upstream: "http://127.0.0.1:5601/", host: "10.20.203.100"
2020/03/12 15:06:07 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.197.35, server: , request: "GET / HTTP/1.1", upstream: "http://127.0.0.1:5601/", host: "10.20.203.100"
2020/03/12 15:06:08 [error] 6#6: *1 no live upstreams while connecting to upstream, client: 10.20.197.35, server: , request: "GET / HTTP/1.1", upstream: "http://docker-kibana/", host: "10.20.203.100"
2020/03/12 15:06:08 [error] 6#6: *1 no live upstreams while connecting to upstream, client: 10.20.197.35, server: , request: "GET / HTTP/1.1", upstream: "http://docker-kibana/", host: "10.20.203.100"
2020/03/12 15:06:08 [error] 6#6: *1 no live upstreams while connecting to upstream, client: 10.20.197.35, server: , request: "GET / HTTP/1.1", upstream: "http://docker-kibana/", host: "10.20.203.100"
2020/03/12 15:06:08 [error] 6#6: *1 no live upstreams while connecting to upstream, client: 10.20.197.35, server: , request: "GET / HTTP/1.1", upstream: "http://docker-kibana/", host: "10.20.203.100"
2020/03/12 15:06:09 [error] 6#6: *1 no live upstreams while connecting to upstream, client: 10.20.197.35, server: , request: "GET / HTTP/1.1", upstream: "http://docker-kibana/", host: "10.20.203.100"
2020/03/12 15:06:09 [error] 6#6: *1 no live upstreams while connecting to upstream, client: 10.20.197.35, server: , request: "GET / HTTP/1.1", upstream: "http://docker-kibana/", host: "10.20.203.100"
2020/03/12 15:06:09 [error] 6#6: *1 no live upstreams while connecting to upstream, client: 10.20.197.35, server: , request: "GET / HTTP/1.1", upstream: "http://docker-kibana/", host: "10.20.203.100"
2020/03/12 15:06:09 [error] 6#6: *1 no live upstreams while connecting to upstream, client: 10.20.197.35, server: , request: "GET / HTTP/1.1", upstream: "http://docker-kibana/", host: "10.20.203.100"
2020/03/12 15:06:09 [error] 6#6: *1 no live upstreams while connecting to upstream, client: 10.20.197.35, server: , request: "GET / HTTP/1.1", upstream: "http://docker-kibana/", host: "10.20.203.100"
2020/03/12 15:06:09 [error] 6#6: *1 no live upstreams while connecting to upstream, client: 10.20.197.35, server: , request: "GET / HTTP/1.1", upstream: "http://docker-kibana/", host: "10.20.203.100"
2020/03/12 15:06:09 [error] 6#6: *1 no live upstreams while connecting to upstream, client: 10.20.197.35, server: , request: "GET / HTTP/1.1", upstream: "http://docker-kibana/", host: "10.20.203.100"
2020/03/12 15:06:11 [error] 6#6: *1 no live upstreams while connecting to upstream, client: 10.20.197.35, server: , request: "GET / HTTP/1.1", upstream: "http://docker-kibana/", host: "10.20.203.100"
2020/03/12 15:06:11 [error] 6#6: *1 no live upstreams while connecting to upstream, client: 10.20.197.35, server: , request: "GET / HTTP/1.1", upstream: "http://docker-kibana/", host: "10.20.203.100"
2020/03/12 15:06:12 [error] 6#6: *1 no live upstreams while connecting to upstream, client: 10.20.197.35, server: , request: "GET / HTTP/1.1", upstream: "http://docker-kibana/", host: "10.20.203.100"
2020/03/12 15:50:22 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.197.35, server: , request: "GET / HTTP/1.1", upstream: "http://127.0.0.1:5601/", host: "10.20.203.100"
2020/03/12 15:50:22 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.197.35, server: , request: "GET / HTTP/1.1", upstream: "http://127.0.0.1:5601/", host: "10.20.203.100"
2020/03/12 15:50:23 [error] 6#6: *1 no live upstreams while connecting to upstream, client: 10.20.197.35, server: , request: "GET / HTTP/1.1", upstream: "http://docker-kibana/", host: "10.20.203.100"
2020/03/12 15:51:28 [error] 6#6: *4 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.197.35, server: , request: "GET / HTTP/1.1", upstream: "http://127.0.0.1:5601/", host: "10.20.203.100"
2020/03/12 15:51:28 [error] 6#6: *4 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.197.35, server: , request: "GET / HTTP/1.1", upstream: "http://127.0.0.1:5601/", host: "10.20.203.100"
2020/03/12 15:51:30 [error] 6#6: *4 no live upstreams while connecting to upstream, client: 10.20.197.35, server: , request: "GET / HTTP/1.1", upstream: "http://docker-kibana/", host: "10.20.203.100"
2020/03/12 15:51:41 [error] 6#6: *4 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.197.35, server: , request: "GET / HTTP/1.1", upstream: "http://127.0.0.1:5601/", host: "10.20.203.100"
2020/03/12 15:51:41 [error] 6#6: *4 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.197.35, server: , request: "GET / HTTP/1.1", upstream: "http://127.0.0.1:5601/", host: "10.20.203.100"
2020/03/12 15:51:42 [error] 6#6: *4 no live upstreams while connecting to upstream, client: 10.20.197.35, server: , request: "GET / HTTP/1.1", upstream: "http://docker-kibana/", host: "10.20.203.100"
2020/03/12 15:51:43 [error] 6#6: *4 no live upstreams while connecting to upstream, client: 10.20.197.35, server: , request: "GET / HTTP/1.1", upstream: "http://docker-kibana/", host: "10.20.203.100"
2020/03/12 15:51:43 [error] 6#6: *4 no live upstreams while connecting to upstream, client: 10.20.197.35, server: , request: "GET / HTTP/1.1", upstream: "http://docker-kibana/", host: "10.20.203.100"
2020/03/12 15:51:43 [error] 6#6: *4 no live upstreams while connecting to upstream, client: 10.20.197.35, server: , request: "GET / HTTP/1.1", upstream: "http://docker-kibana/", host: "10.20.203.100"
2020/03/12 15:51:43 [error] 6#6: *4 no live upstreams while connecting to upstream, client: 10.20.197.35, server: , request: "GET / HTTP/1.1", upstream: "http://docker-kibana/", host: "10.20.203.100"
2020/03/12 15:51:44 [error] 6#6: *4 no live upstreams while connecting to upstream, client: 10.20.197.35, server: , request: "GET / HTTP/1.1", upstream: "http://docker-kibana/", host: "10.20.203.100"
2020/03/12 15:51:45 [error] 6#6: *4 no live upstreams while connecting to upstream, client: 10.20.197.35, server: , request: "GET / HTTP/1.1", upstream: "http://docker-kibana/", host: "10.20.203.100"
2020/03/12 15:51:47 [error] 6#6: *4 no live upstreams while connecting to upstream, client: 10.20.197.35, server: , request: "GET / HTTP/1.1", upstream: "http://docker-kibana/", host: "10.20.203.100"
2020/03/12 15:51:54 [error] 6#6: *4 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.197.35, server: , request: "GET / HTTP/1.1", upstream: "http://127.0.0.1:5601/", host: "10.20.203.100"
2020/03/12 15:51:54 [error] 6#6: *4 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.197.35, server: , request: "GET / HTTP/1.1", upstream: "http://127.0.0.1:5601/", host: "10.20.203.100"
2020/03/12 15:51:55 [error] 6#6: *4 no live upstreams while connecting to upstream, client: 10.20.197.35, server: , request: "GET / HTTP/1.1", upstream: "http://docker-kibana/", host: "10.20.203.100"
2020/03/12 15:52:07 [error] 6#6: *4 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.197.35, server: , request: "GET / HTTP/1.1", upstream: "http://127.0.0.1:5601/", host: "10.20.203.100"
2020/03/12 15:52:07 [error] 6#6: *4 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.197.35, server: , request: "GET / HTTP/1.1", upstream: "http://127.0.0.1:5601/", host: "10.20.203.100"
2020/03/12 15:52:20 [error] 6#6: *4 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.197.35, server: , request: "GET / HTTP/1.1", upstream: "http://127.0.0.1:5601/", host: "10.20.203.100"
2020/03/12 15:52:20 [error] 6#6: *4 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.197.35, server: , request: "GET / HTTP/1.1", upstream: "http://127.0.0.1:5601/", host: "10.20.203.100"
2020/03/12 15:52:22 [error] 6#6: *4 no live upstreams while connecting to upstream, client: 10.20.197.35, server: , request: "GET / HTTP/1.1", upstream: "http://docker-kibana/", host: "10.20.203.100"
2020/03/12 16:08:45 [error] 6#6: *15 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.197.35, server: , request: "GET / HTTP/1.1", upstream: "http://127.0.0.1:5601/", host: "10.20.203.100"
2020/03/12 16:08:45 [error] 6#6: *15 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.197.35, server: , request: "GET / HTTP/1.1", upstream: "http://127.0.0.1:5601/", host: "10.20.203.100"
2020/03/12 16:08:47 [error] 6#6: *15 no live upstreams while connecting to upstream, client: 10.20.197.35, server: , request: "GET / HTTP/1.1", upstream: "http://docker-kibana/", host: "10.20.203.100"
2020/03/12 16:08:48 [error] 6#6: *15 no live upstreams while connecting to upstream, client: 10.20.197.35, server: , request: "GET / HTTP/1.1", upstream: "http://docker-kibana/", host: "10.20.203.100"
2020/03/12 16:08:48 [error] 6#6: *15 no live upstreams while connecting to upstream, client: 10.20.197.35, server: , request: "GET / HTTP/1.1", upstream: "http://docker-kibana/", host: "10.20.203.100"
2020/03/12 16:08:48 [error] 6#6: *15 no live upstreams while connecting to upstream, client: 10.20.197.35, server: , request: "GET / HTTP/1.1", upstream: "http://docker-kibana/", host: "10.20.203.100"
2020/03/12 16:08:48 [error] 6#6: *15 no live upstreams while connecting to upstream, client: 10.20.197.35, server: , request: "GET / HTTP/1.1", upstream: "http://docker-kibana/", host: "10.20.203.100"
2020/03/12 16:12:05 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.197.35, server: , request: "GET / HTTP/1.1", upstream: "http://127.0.0.1:5601/", host: "10.20.203.100"
2020/03/12 16:12:05 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.197.35, server: , request: "GET / HTTP/1.1", upstream: "http://127.0.0.1:5601/", host: "10.20.203.100"
2020/03/12 16:18:32 [error] 6#6: *4 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.197.35, server: , request: "GET / HTTP/1.1", upstream: "http://127.0.0.1:5601/", host: "10.20.203.100"
2020/03/12 16:18:32 [error] 6#6: *4 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.197.35, server: , request: "GET / HTTP/1.1", upstream: "http://127.0.0.1:5601/", host: "10.20.203.100"
2020/03/12 16:18:42 [error] 6#6: *4 no live upstreams while connecting to upstream, client: 10.20.197.35, server: , request: "GET / HTTP/1.1", upstream: "http://docker-kibana/", host: "10.20.203.100"
2020/03/12 16:18:42 [error] 6#6: *4 no live upstreams while connecting to upstream, client: 10.20.197.35, server: , request: "GET / HTTP/1.1", upstream: "http://docker-kibana/", host: "10.20.203.100"
2020/03/12 16:18:42 [error] 6#6: *4 no live upstreams while connecting to upstream, client: 10.20.197.35, server: , request: "GET / HTTP/1.1", upstream: "http://docker-kibana/", host: "10.20.203.100"
2020/03/12 16:26:51 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.197.35, server: , request: "GET / HTTP/1.1", upstream: "http://127.0.0.1:5601/", host: "10.20.203.100"
2020/03/12 16:26:51 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.197.35, server: , request: "GET / HTTP/1.1", upstream: "http://127.0.0.1:5601/", host: "10.20.203.100"
2020/03/12 16:26:52 [error] 6#6: *1 no live upstreams while connecting to upstream, client: 10.20.197.35, server: , request: "GET / HTTP/1.1", upstream: "http://docker-kibana/", host: "10.20.203.100"
2020/03/13 09:32:47 [error] 6#6: *4 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.197.35, server: , request: "GET / HTTP/1.1", upstream: "http://127.0.0.1:5601/", host: "10.20.203.100"
2020/03/13 09:32:47 [error] 6#6: *4 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.197.35, server: , request: "GET / HTTP/1.1", upstream: "http://127.0.0.1:5601/", host: "10.20.203.100"
2020/03/13 09:32:48 [error] 6#6: *4 no live upstreams while connecting to upstream, client: 10.20.197.35, server: , request: "GET / HTTP/1.1", upstream: "http://docker-kibana/", host: "10.20.203.100"
2020/03/13 09:32:49 [error] 6#6: *4 no live upstreams while connecting to upstream, client: 10.20.197.35, server: , request: "GET / HTTP/1.1", upstream: "http://docker-kibana/", host: "10.20.203.100"
2020/03/13 09:32:49 [error] 6#6: *4 no live upstreams while connecting to upstream, client: 10.20.197.35, server: , request: "GET / HTTP/1.1", upstream: "http://docker-kibana/", host: "10.20.203.100"
2020/03/13 09:32:49 [error] 6#6: *4 no live upstreams while connecting to upstream, client: 10.20.197.35, server: , request: "GET / HTTP/1.1", upstream: "http://docker-kibana/", host: "10.20.203.100"
2020/03/13 09:32:49 [error] 6#6: *4 no live upstreams while connecting to upstream, client: 10.20.197.35, server: , request: "GET / HTTP/1.1", upstream: "http://docker-kibana/", host: "10.20.203.100"
2020/03/13 09:32:49 [error] 6#6: *4 no live upstreams while connecting to upstream, client: 10.20.197.35, server: , request: "GET / HTTP/1.1", upstream: "http://docker-kibana/", host: "10.20.203.100"
2020/03/13 09:32:49 [error] 6#6: *4 no live upstreams while connecting to upstream, client: 10.20.197.35, server: , request: "GET / HTTP/1.1", upstream: "http://docker-kibana/", host: "10.20.203.100"
2020/03/13 11:06:10 [error] 6#6: *7 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.197.35, server: , request: "GET / HTTP/1.1", upstream: "http://127.0.0.1:5601/", host: "10.20.203.100"
2020/03/13 11:06:10 [error] 6#6: *7 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.197.35, server: , request: "GET / HTTP/1.1", upstream: "http://127.0.0.1:5601/", host: "10.20.203.100"
2020/03/13 14:16:20 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.197.35, server: , request: "GET / HTTP/1.1", upstream: "http://127.0.0.1:5601/", host: "10.20.203.100"
2020/03/13 14:16:20 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.197.35, server: , request: "GET / HTTP/1.1", upstream: "http://127.0.0.1:5601/", host: "10.20.203.100"
2020/03/13 14:16:22 [error] 6#6: *1 no live upstreams while connecting to upstream, client: 10.20.197.35, server: , request: "GET / HTTP/1.1", upstream: "http://docker-kibana/", host: "10.20.203.100"
2020/03/13 14:16:33 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.197.35, server: , request: "GET / HTTP/1.1", upstream: "http://127.0.0.1:5601/", host: "10.20.203.100"
2020/03/13 14:16:33 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.197.35, server: , request: "GET / HTTP/1.1", upstream: "http://127.0.0.1:5601/", host: "10.20.203.100"
2020/03/18 10:23:17 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.197.129, server: , request: "GET / HTTP/1.1", upstream: "http://127.0.0.1:5601/", host: "10.20.203.100"
2020/03/18 10:23:17 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.197.129, server: , request: "GET / HTTP/1.1", upstream: "http://127.0.0.1:5601/", host: "10.20.203.100"
2020/03/18 10:23:19 [error] 6#6: *1 no live upstreams while connecting to upstream, client: 10.20.197.129, server: , request: "GET / HTTP/1.1", upstream: "http://docker-kibana/", host: "10.20.203.100"
2020/03/18 10:36:38 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.197.129, server: , request: "GET / HTTP/1.1", upstream: "http://127.0.0.1:5601/", host: "10.20.203.100"
2020/03/18 10:36:38 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.197.129, server: , request: "GET / HTTP/1.1", upstream: "http://127.0.0.1:5601/", host: "10.20.203.100"
2020/03/18 10:37:00 [error] 6#6: *4 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.197.129, server: , request: "GET / HTTP/1.1", upstream: "http://127.0.0.1:5601/", host: "10.20.203.100"
2020/03/18 10:37:00 [error] 6#6: *4 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.197.129, server: , request: "GET / HTTP/1.1", upstream: "http://127.0.0.1:5601/", host: "10.20.203.100"
2020/03/18 10:37:16 [error] 6#6: *7 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.197.129, server: , request: "GET / HTTP/1.1", upstream: "http://127.0.0.1:5601/", host: "10.20.203.100"
2020/03/18 10:37:16 [error] 6#6: *7 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.197.129, server: , request: "GET / HTTP/1.1", upstream: "http://127.0.0.1:5601/", host: "10.20.203.100"
2020/03/18 10:37:16 [error] 6#6: *7 no live upstreams while connecting to upstream, client: 10.20.197.129, server: , request: "GET / HTTP/1.1", upstream: "http://docker-kibana/", host: "10.20.203.100"
2020/03/18 10:37:16 [error] 6#6: *10 no live upstreams while connecting to upstream, client: 10.20.197.129, server: , request: "GET / HTTP/1.1", upstream: "http://docker-kibana/", host: "10.20.203.100"
2020/03/18 10:37:17 [error] 6#6: *10 no live upstreams while connecting to upstream, client: 10.20.197.129, server: , request: "GET / HTTP/1.1", upstream: "http://docker-kibana/", host: "10.20.203.100"
2020/03/18 10:37:17 [error] 6#6: *10 no live upstreams while connecting to upstream, client: 10.20.197.129, server: , request: "GET / HTTP/1.1", upstream: "http://docker-kibana/", host: "10.20.203.100"
2020/03/18 10:37:17 [error] 6#6: *10 no live upstreams while connecting to upstream, client: 10.20.197.129, server: , request: "GET / HTTP/1.1", upstream: "http://docker-kibana/", host: "10.20.203.100"
2020/03/18 10:37:18 [error] 6#6: *10 no live upstreams while connecting to upstream, client: 10.20.197.129, server: , request: "GET / HTTP/1.1", upstream: "http://docker-kibana/", host: "10.20.203.100"
2020/03/18 10:37:18 [error] 6#6: *10 no live upstreams while connecting to upstream, client: 10.20.197.129, server: , request: "GET / HTTP/1.1", upstream: "http://docker-kibana/", host: "10.20.203.100"
2020/03/18 10:37:18 [error] 6#6: *10 no live upstreams while connecting to upstream, client: 10.20.197.129, server: , request: "GET / HTTP/1.1", upstream: "http://docker-kibana/", host: "10.20.203.100"
2020/03/18 10:37:18 [error] 6#6: *10 no live upstreams while connecting to upstream, client: 10.20.197.129, server: , request: "GET / HTTP/1.1", upstream: "http://docker-kibana/", host: "10.20.203.100"
2020/03/18 10:37:18 [error] 6#6: *10 no live upstreams while connecting to upstream, client: 10.20.197.129, server: , request: "GET / HTTP/1.1", upstream: "http://docker-kibana/", host: "10.20.203.100"
2020/03/18 10:37:21 [error] 6#6: *10 no live upstreams while connecting to upstream, client: 10.20.197.129, server: , request: "GET / HTTP/1.1", upstream: "http://docker-kibana/", host: "10.20.203.100"
2020/03/18 10:37:22 [error] 6#6: *10 no live upstreams while connecting to upstream, client: 10.20.197.129, server: , request: "GET / HTTP/1.1", upstream: "http://docker-kibana/", host: "10.20.203.100"
2020/03/18 10:37:38 [error] 6#6: *10 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.197.129, server: , request: "GET / HTTP/1.1", upstream: "http://127.0.0.1:5601/", host: "10.20.203.100"
2020/03/18 10:37:38 [error] 6#6: *10 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.197.129, server: , request: "GET / HTTP/1.1", upstream: "http://127.0.0.1:5601/", host: "10.20.203.100"
2020/03/18 10:37:39 [error] 6#6: *13 no live upstreams while connecting to upstream, client: 10.20.197.129, server: , request: "GET / HTTP/1.1", upstream: "http://docker-kibana/", host: "10.20.203.100"
2020/03/18 10:37:39 [error] 6#6: *13 no live upstreams while connecting to upstream, client: 10.20.197.129, server: , request: "GET / HTTP/1.1", upstream: "http://docker-kibana/", host: "10.20.203.100"
2020/03/18 10:37:39 [error] 6#6: *13 no live upstreams while connecting to upstream, client: 10.20.197.129, server: , request: "GET / HTTP/1.1", upstream: "http://docker-kibana/", host: "10.20.203.100"
2020/03/18 10:37:40 [error] 6#6: *13 no live upstreams while connecting to upstream, client: 10.20.197.129, server: , request: "GET / HTTP/1.1", upstream: "http://docker-kibana/", host: "10.20.203.100"
2020/03/18 10:37:44 [error] 6#6: *13 no live upstreams while connecting to upstream, client: 10.20.197.129, server: , request: "GET / HTTP/1.1", upstream: "http://docker-kibana/", host: "10.20.203.100"
2020/03/18 10:37:44 [error] 6#6: *14 no live upstreams while connecting to upstream, client: 10.20.197.129, server: , request: "GET / HTTP/1.1", upstream: "http://docker-kibana/", host: "10.20.203.100"
2020/03/18 10:37:44 [error] 6#6: *14 no live upstreams while connecting to upstream, client: 10.20.197.129, server: , request: "GET / HTTP/1.1", upstream: "http://docker-kibana/", host: "10.20.203.100"
2020/03/18 10:37:49 [error] 6#6: *14 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.197.129, server: , request: "GET / HTTP/1.1", upstream: "http://127.0.0.1:5601/", host: "10.20.203.100"
2020/03/18 10:37:49 [error] 6#6: *14 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.197.129, server: , request: "GET / HTTP/1.1", upstream: "http://127.0.0.1:5601/", host: "10.20.203.100"
2020/03/18 10:37:49 [error] 6#6: *17 no live upstreams while connecting to upstream, client: 10.20.197.129, server: , request: "GET / HTTP/1.1", upstream: "http://docker-kibana/", host: "10.20.203.100"
2020/03/18 10:37:49 [error] 6#6: *17 no live upstreams while connecting to upstream, client: 10.20.197.129, server: , request: "GET / HTTP/1.1", upstream: "http://docker-kibana/", host: "10.20.203.100"
2020/03/18 10:37:50 [error] 6#6: *17 no live upstreams while connecting to upstream, client: 10.20.197.129, server: , request: "GET / HTTP/1.1", upstream: "http://docker-kibana/", host: "10.20.203.100"
2020/03/18 10:37:55 [error] 6#6: *18 no live upstreams while connecting to upstream, client: 10.20.197.129, server: , request: "GET / HTTP/1.1", upstream: "http://docker-kibana/", host: "10.20.203.100"
2020/03/18 10:37:56 [error] 6#6: *18 no live upstreams while connecting to upstream, client: 10.20.197.129, server: , request: "GET / HTTP/1.1", upstream: "http://docker-kibana/", host: "10.20.203.100"
2020/03/18 10:37:56 [error] 6#6: *18 no live upstreams while connecting to upstream, client: 10.20.197.129, server: , request: "GET / HTTP/1.1", upstream: "http://docker-kibana/", host: "10.20.203.100"
2020/03/18 10:37:56 [error] 6#6: *18 no live upstreams while connecting to upstream, client: 10.20.197.129, server: , request: "GET / HTTP/1.1", upstream: "http://docker-kibana/", host: "10.20.203.100"
2020/03/18 10:37:57 [error] 6#6: *19 no live upstreams while connecting to upstream, client: 10.20.197.129, server: , request: "GET / HTTP/1.1", upstream: "http://docker-kibana/", host: "10.20.203.100"
2020/03/18 10:38:00 [error] 6#6: *19 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.197.129, server: , request: "GET / HTTP/1.1", upstream: "http://127.0.0.1:5601/", host: "10.20.203.100"
2020/03/18 10:38:00 [error] 6#6: *19 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.197.129, server: , request: "GET / HTTP/1.1", upstream: "http://127.0.0.1:5601/", host: "10.20.203.100"
2020/03/18 10:38:00 [error] 6#6: *19 no live upstreams while connecting to upstream, client: 10.20.197.129, server: , request: "GET / HTTP/1.1", upstream: "http://docker-kibana/", host: "10.20.203.100"
2020/03/18 10:38:00 [error] 6#6: *19 no live upstreams while connecting to upstream, client: 10.20.197.129, server: , request: "GET / HTTP/1.1", upstream: "http://docker-kibana/", host: "10.20.203.100"
2020/03/18 10:38:01 [error] 6#6: *19 no live upstreams while connecting to upstream, client: 10.20.197.129, server: , request: "GET / HTTP/1.1", upstream: "http://docker-kibana/", host: "10.20.203.100"
2020/03/18 10:38:01 [error] 6#6: *22 no live upstreams while connecting to upstream, client: 10.20.197.129, server: , request: "GET / HTTP/1.1", upstream: "http://docker-kibana/", host: "10.20.203.100"
2020/03/18 10:38:01 [error] 6#6: *22 no live upstreams while connecting to upstream, client: 10.20.197.129, server: , request: "GET / HTTP/1.1", upstream: "http://docker-kibana/", host: "10.20.203.100"
2020/03/18 10:38:01 [error] 6#6: *22 no live upstreams while connecting to upstream, client: 10.20.197.129, server: , request: "GET / HTTP/1.1", upstream: "http://docker-kibana/", host: "10.20.203.100"
2020/03/18 10:38:02 [error] 6#6: *22 no live upstreams while connecting to upstream, client: 10.20.197.129, server: , request: "GET / HTTP/1.1", upstream: "http://docker-kibana/", host: "10.20.203.100"
2020/03/18 10:38:02 [error] 6#6: *22 no live upstreams while connecting to upstream, client: 10.20.197.129, server: , request: "GET / HTTP/1.1", upstream: "http://docker-kibana/", host: "10.20.203.100"
2020/03/18 10:38:02 [error] 6#6: *22 no live upstreams while connecting to upstream, client: 10.20.197.129, server: , request: "GET / HTTP/1.1", upstream: "http://docker-kibana/", host: "10.20.203.100"
2020/03/18 10:38:02 [error] 6#6: *22 no live upstreams while connecting to upstream, client: 10.20.197.129, server: , request: "GET / HTTP/1.1", upstream: "http://docker-kibana/", host: "10.20.203.100"
2020/03/18 10:38:02 [error] 6#6: *22 no live upstreams while connecting to upstream, client: 10.20.197.129, server: , request: "GET / HTTP/1.1", upstream: "http://docker-kibana/", host: "10.20.203.100"
2020/03/18 10:38:02 [error] 6#6: *22 no live upstreams while connecting to upstream, client: 10.20.197.129, server: , request: "GET / HTTP/1.1", upstream: "http://docker-kibana/", host: "10.20.203.100"
2020/03/18 10:38:03 [error] 6#6: *22 no live upstreams while connecting to upstream, client: 10.20.197.129, server: , request: "GET / HTTP/1.1", upstream: "http://docker-kibana/", host: "10.20.203.100"
2020/03/18 10:38:03 [error] 6#6: *22 no live upstreams while connecting to upstream, client: 10.20.197.129, server: , request: "GET / HTTP/1.1", upstream: "http://docker-kibana/", host: "10.20.203.100"
2020/03/18 10:38:04 [error] 6#6: *22 no live upstreams while connecting to upstream, client: 10.20.197.129, server: , request: "GET / HTTP/1.1", upstream: "http://docker-kibana/", host: "10.20.203.100"
2020/03/18 10:38:05 [error] 6#6: *22 no live upstreams while connecting to upstream, client: 10.20.197.129, server: , request: "GET / HTTP/1.1", upstream: "http://docker-kibana/", host: "10.20.203.100"
2020/03/18 10:38:05 [error] 6#6: *22 no live upstreams while connecting to upstream, client: 10.20.197.129, server: , request: "GET / HTTP/1.1", upstream: "http://docker-kibana/", host: "10.20.203.100"
2020/03/18 10:38:05 [error] 6#6: *22 no live upstreams while connecting to upstream, client: 10.20.197.129, server: , request: "GET / HTTP/1.1", upstream: "http://docker-kibana/", host: "10.20.203.100"
2020/03/18 10:38:06 [error] 6#6: *23 no live upstreams while connecting to upstream, client: 10.20.197.129, server: , request: "GET / HTTP/1.1", upstream: "http://docker-kibana/", host: "10.20.203.100"
2020/03/18 10:38:07 [error] 6#6: *23 no live upstreams while connecting to upstream, client: 10.20.197.129, server: , request: "GET / HTTP/1.1", upstream: "http://docker-kibana/", host: "10.20.203.100"
2020/03/18 10:38:07 [error] 6#6: *23 no live upstreams while connecting to upstream, client: 10.20.197.129, server: , request: "GET / HTTP/1.1", upstream: "http://docker-kibana/", host: "10.20.203.100"
2020/03/18 10:38:08 [error] 6#6: *23 no live upstreams while connecting to upstream, client: 10.20.197.129, server: , request: "GET / HTTP/1.1", upstream: "http://docker-kibana/", host: "10.20.203.100"
2020/03/18 10:38:08 [error] 6#6: *23 no live upstreams while connecting to upstream, client: 10.20.197.129, server: , request: "GET / HTTP/1.1", upstream: "http://docker-kibana/", host: "10.20.203.100"
2020/03/18 10:38:11 [error] 6#6: *23 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.197.129, server: , request: "GET / HTTP/1.1", upstream: "http://127.0.0.1:5601/", host: "10.20.203.100"
2020/03/18 10:38:11 [error] 6#6: *23 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.197.129, server: , request: "GET / HTTP/1.1", upstream: "http://127.0.0.1:5601/", host: "10.20.203.100"
2020/03/18 10:38:11 [error] 6#6: *23 no live upstreams while connecting to upstream, client: 10.20.197.129, server: , request: "GET / HTTP/1.1", upstream: "http://docker-kibana/", host: "10.20.203.100"
2020/03/20 14:23:40 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.197.129, server: , request: "GET / HTTP/1.1", upstream: "http://127.0.0.1:5601/", host: "10.20.203.100"
2020/03/20 14:23:40 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.197.129, server: , request: "GET / HTTP/1.1", upstream: "http://127.0.0.1:5601/", host: "10.20.203.100"
2020/03/20 14:23:42 [error] 6#6: *1 no live upstreams while connecting to upstream, client: 10.20.197.129, server: , request: "GET / HTTP/1.1", upstream: "http://docker-kibana/", host: "10.20.203.100"
2020/03/20 14:23:43 [error] 6#6: *1 no live upstreams while connecting to upstream, client: 10.20.197.129, server: , request: "GET / HTTP/1.1", upstream: "http://docker-kibana/", host: "10.20.203.100"
2020/03/20 14:23:43 [error] 6#6: *1 no live upstreams while connecting to upstream, client: 10.20.197.129, server: , request: "GET / HTTP/1.1", upstream: "http://docker-kibana/", host: "10.20.203.100"
2020/03/20 14:23:43 [error] 6#6: *1 no live upstreams while connecting to upstream, client: 10.20.197.129, server: , request: "GET / HTTP/1.1", upstream: "http://docker-kibana/", host: "10.20.203.100"
2020/03/20 14:23:43 [error] 6#6: *1 no live upstreams while connecting to upstream, client: 10.20.197.129, server: , request: "GET / HTTP/1.1", upstream: "http://docker-kibana/", host: "10.20.203.100"
2020/03/20 14:23:43 [error] 6#6: *1 no live upstreams while connecting to upstream, client: 10.20.197.129, server: , request: "GET / HTTP/1.1", upstream: "http://docker-kibana/", host: "10.20.203.100"
2020/03/20 14:23:44 [error] 6#6: *1 no live upstreams while connecting to upstream, client: 10.20.197.129, server: , request: "GET / HTTP/1.1", upstream: "http://docker-kibana/", host: "10.20.203.100"
2020/03/20 14:23:44 [error] 6#6: *1 no live upstreams while connecting to upstream, client: 10.20.197.129, server: , request: "GET / HTTP/1.1", upstream: "http://docker-kibana/", host: "10.20.203.100"
2020/03/20 14:23:44 [error] 6#6: *1 no live upstreams while connecting to upstream, client: 10.20.197.129, server: , request: "GET / HTTP/1.1", upstream: "http://docker-kibana/", host: "10.20.203.100"
2020/03/20 14:23:44 [error] 6#6: *1 no live upstreams while connecting to upstream, client: 10.20.197.129, server: , request: "GET / HTTP/1.1", upstream: "http://docker-kibana/", host: "10.20.203.100"
2020/03/20 14:23:45 [error] 6#6: *1 no live upstreams while connecting to upstream, client: 10.20.197.129, server: , request: "GET / HTTP/1.1", upstream: "http://docker-kibana/", host: "10.20.203.100"
2020/03/20 14:23:45 [error] 6#6: *1 no live upstreams while connecting to upstream, client: 10.20.197.129, server: , request: "GET / HTTP/1.1", upstream: "http://docker-kibana/", host: "10.20.203.100"
2020/03/20 14:23:48 [error] 6#6: *1 no live upstreams while connecting to upstream, client: 10.20.197.129, server: , request: "GET / HTTP/1.1", upstream: "http://docker-kibana/", host: "10.20.203.100"
2020/03/20 14:23:48 [error] 6#6: *1 no live upstreams while connecting to upstream, client: 10.20.197.129, server: , request: "GET / HTTP/1.1", upstream: "http://docker-kibana/", host: "10.20.203.100"
2020/03/20 14:23:48 [error] 6#6: *1 no live upstreams while connecting to upstream, client: 10.20.197.129, server: , request: "GET / HTTP/1.1", upstream: "http://docker-kibana/", host: "10.20.203.100"
2020/03/20 14:23:49 [error] 6#6: *1 no live upstreams while connecting to upstream, client: 10.20.197.129, server: , request: "GET / HTTP/1.1", upstream: "http://docker-kibana/", host: "10.20.203.100"
2020/03/20 14:23:49 [error] 6#6: *1 no live upstreams while connecting to upstream, client: 10.20.197.129, server: , request: "GET / HTTP/1.1", upstream: "http://docker-kibana/", host: "10.20.203.100"
2020/03/20 14:23:49 [error] 6#6: *1 no live upstreams while connecting to upstream, client: 10.20.197.129, server: , request: "GET / HTTP/1.1", upstream: "http://docker-kibana/", host: "10.20.203.100"
2020/03/20 14:23:50 [error] 6#6: *1 no live upstreams while connecting to upstream, client: 10.20.197.129, server: , request: "GET / HTTP/1.1", upstream: "http://docker-kibana/", host: "10.20.203.100"
2020/03/20 14:23:50 [error] 6#6: *1 no live upstreams while connecting to upstream, client: 10.20.197.129, server: , request: "GET / HTTP/1.1", upstream: "http://docker-kibana/", host: "10.20.203.100"
2020/03/20 14:23:50 [error] 6#6: *1 no live upstreams while connecting to upstream, client: 10.20.197.129, server: , request: "GET / HTTP/1.1", upstream: "http://docker-kibana/", host: "10.20.203.100"
2020/03/20 14:23:50 [error] 6#6: *1 no live upstreams while connecting to upstream, client: 10.20.197.129, server: , request: "GET / HTTP/1.1", upstream: "http://docker-kibana/", host: "10.20.203.100"
2020/03/20 14:23:51 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.197.129, server: , request: "GET / HTTP/1.1", upstream: "http://127.0.0.1:5601/", host: "10.20.203.100"
2020/03/20 14:23:51 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.197.129, server: , request: "GET / HTTP/1.1", upstream: "http://127.0.0.1:5601/", host: "10.20.203.100"
2020/03/20 14:23:51 [error] 6#6: *1 no live upstreams while connecting to upstream, client: 10.20.197.129, server: , request: "GET / HTTP/1.1", upstream: "http://docker-kibana/", host: "10.20.203.100"
2020/03/20 14:23:51 [error] 6#6: *1 no live upstreams while connecting to upstream, client: 10.20.197.129, server: , request: "GET / HTTP/1.1", upstream: "http://docker-kibana/", host: "10.20.203.100"
2020/03/20 14:23:51 [error] 6#6: *1 no live upstreams while connecting to upstream, client: 10.20.197.129, server: , request: "GET / HTTP/1.1", upstream: "http://docker-kibana/", host: "10.20.203.100"
2020/03/20 14:23:52 [error] 6#6: *1 no live upstreams while connecting to upstream, client: 10.20.197.129, server: , request: "GET / HTTP/1.1", upstream: "http://docker-kibana/", host: "10.20.203.100"
2020/03/20 14:23:53 [error] 6#6: *1 no live upstreams while connecting to upstream, client: 10.20.197.129, server: , request: "GET / HTTP/1.1", upstream: "http://docker-kibana/", host: "10.20.203.100"
2020/03/20 14:23:53 [error] 6#6: *6 no live upstreams while connecting to upstream, client: 10.20.197.129, server: , request: "GET / HTTP/1.1", upstream: "http://docker-kibana/", host: "10.20.203.100"
2020/03/20 14:25:01 [error] 6#6: *6 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.197.129, server: , request: "GET / HTTP/1.1", upstream: "http://127.0.0.1:5601/", host: "10.20.203.100"
2020/03/20 14:25:01 [error] 6#6: *6 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.197.129, server: , request: "GET / HTTP/1.1", upstream: "http://127.0.0.1:5601/", host: "10.20.203.100"
2020/03/20 14:25:03 [error] 6#6: *6 no live upstreams while connecting to upstream, client: 10.20.197.129, server: , request: "GET / HTTP/1.1", upstream: "http://docker-kibana/", host: "10.20.203.100"
2020/03/20 14:25:03 [error] 6#6: *6 no live upstreams while connecting to upstream, client: 10.20.197.129, server: , request: "GET / HTTP/1.1", upstream: "http://docker-kibana/", host: "10.20.203.100"
2020/03/20 14:25:03 [error] 6#6: *6 no live upstreams while connecting to upstream, client: 10.20.197.129, server: , request: "GET / HTTP/1.1", upstream: "http://docker-kibana/", host: "10.20.203.100"
2020/03/20 14:25:04 [error] 6#6: *6 no live upstreams while connecting to upstream, client: 10.20.197.129, server: , request: "GET / HTTP/1.1", upstream: "http://docker-kibana/", host: "10.20.203.100"
2020/03/20 14:25:04 [error] 6#6: *6 no live upstreams while connecting to upstream, client: 10.20.197.129, server: , request: "GET / HTTP/1.1", upstream: "http://docker-kibana/", host: "10.20.203.100"
2020/03/20 14:25:04 [error] 6#6: *6 no live upstreams while connecting to upstream, client: 10.20.197.129, server: , request: "GET / HTTP/1.1", upstream: "http://docker-kibana/", host: "10.20.203.100"
2020/03/20 14:25:04 [error] 6#6: *6 no live upstreams while connecting to upstream, client: 10.20.197.129, server: , request: "GET / HTTP/1.1", upstream: "http://docker-kibana/", host: "10.20.203.100"
2020/03/20 14:25:05 [error] 6#6: *6 no live upstreams while connecting to upstream, client: 10.20.197.129, server: , request: "GET / HTTP/1.1", upstream: "http://docker-kibana/", host: "10.20.203.100"
2020/03/20 14:25:05 [error] 6#6: *6 no live upstreams while connecting to upstream, client: 10.20.197.129, server: , request: "GET / HTTP/1.1", upstream: "http://docker-kibana/", host: "10.20.203.100"
2020/03/20 14:25:05 [error] 6#6: *6 no live upstreams while connecting to upstream, client: 10.20.197.129, server: , request: "GET / HTTP/1.1", upstream: "http://docker-kibana/", host: "10.20.203.100"
2020/03/20 14:25:05 [error] 6#6: *6 no live upstreams while connecting to upstream, client: 10.20.197.129, server: , request: "GET / HTTP/1.1", upstream: "http://docker-kibana/", host: "10.20.203.100"
2020/03/20 14:25:05 [error] 6#6: *6 no live upstreams while connecting to upstream, client: 10.20.197.129, server: , request: "GET / HTTP/1.1", upstream: "http://docker-kibana/", host: "10.20.203.100"
2020/03/20 14:29:05 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.197.129, server: , request: "GET / HTTP/1.1", upstream: "http://127.0.0.1:5601/", host: "10.20.203.100"
2020/03/20 14:29:05 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.197.129, server: , request: "GET / HTTP/1.1", upstream: "http://127.0.0.1:5601/", host: "10.20.203.100"
2020/03/20 14:30:19 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.197.129, server: , request: "GET / HTTP/1.1", upstream: "http://127.0.0.1:5601/", host: "10.20.203.100"
2020/03/20 14:30:19 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 10.20.197.129, server: , request: "GET / HTTP/1.1", upstream: "http://127.0.0.1:5601/", host: "10.20.203.100"
2020/03/20 14:30:21 [error] 6#6: *1 no live upstreams while connecting to upstream, client: 10.20.197.129, server: , request: "GET / HTTP/1.1", upstream: "http://docker-kibana/", host: "10.20.203.100"
2020/03/20 14:30:21 [error] 6#6: *1 no live upstreams while connecting to upstream, client: 10.20.197.129, server: , request: "GET / HTTP/1.1", upstream: "http://docker-kibana/", host: "10.20.203.100"
2020/03/20 14:30:21 [error] 6#6: *1 no live upstreams while connecting to upstream, client: 10.20.197.129, server: , request: "GET / HTTP/1.1", upstream: "http://docker-kibana/", host: "10.20.203.100"
